<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="org.mozilla.javascript.tests.MozillaSuiteTest" tests="83" skipped="0" failures="15" errors="0" timestamp="2019-01-26T20:33:21" hostname="swami-VirtualBox" time="110.918">
  <properties/>
  <testcase name="runMozillaTest[0, js=testsrc/tests/Rhino_ECMA262_Tests/test_array.js, opt=0]" classname="org.mozilla.javascript.tests.MozillaSuiteTest" time="6.945"/>
  <testcase name="runMozillaTest[1, js=testsrc/tests/Rhino_ECMA262_Tests/test_array_from.js, opt=0]" classname="org.mozilla.javascript.tests.MozillaSuiteTest" time="7.409"/>
  <testcase name="runMozillaTest[2, js=testsrc/tests/Rhino_ECMA262_Tests/test_array_prototype_every.js, opt=0]" classname="org.mozilla.javascript.tests.MozillaSuiteTest" time="5.239"/>
  <testcase name="runMozillaTest[3, js=testsrc/tests/Rhino_ECMA262_Tests/test_array_prototype_filter.js, opt=0]" classname="org.mozilla.javascript.tests.MozillaSuiteTest" time="4.266"/>
  <testcase name="runMozillaTest[4, js=testsrc/tests/Rhino_ECMA262_Tests/test_array_prototype_find.js, opt=0]" classname="org.mozilla.javascript.tests.MozillaSuiteTest" time="4.583"/>
  <testcase name="runMozillaTest[5, js=testsrc/tests/Rhino_ECMA262_Tests/test_array_prototype_findindex.js, opt=0]" classname="org.mozilla.javascript.tests.MozillaSuiteTest" time="4.286"/>
  <testcase name="runMozillaTest[6, js=testsrc/tests/Rhino_ECMA262_Tests/test_array_prototype_foreach.js, opt=0]" classname="org.mozilla.javascript.tests.MozillaSuiteTest" time="5.588"/>
  <testcase name="runMozillaTest[7, js=testsrc/tests/Rhino_ECMA262_Tests/test_array_prototype_includes.js, opt=0]" classname="org.mozilla.javascript.tests.MozillaSuiteTest" time="0.437">
    <failure message="junit.framework.AssertionFailedError: junit.framework.AssertionFailedError: junit.framework.AssertionFailedError: In &quot;testsrc/tests/Rhino_ECMA262_Tests/test_array_prototype_includes.js&quot;:&#10;JavaScript errors in testsrc/tests/Rhino_ECMA262_Tests/test_array_prototype_includes.js:&#10;uncaught JavaScript runtime exception: TypeError: Cannot find function includes in object /\r\n|\r|\n/.&#10;&#9;at junit.framework.Assert.fail(Assert.java:57)&#10;&#9;at org.mozilla.javascript.tests.MozillaSuiteTest$JunitStatus.failed(MozillaSuiteTest.java:159)&#10;&#9;at org.mozilla.javascript.drivers.ShellTest$Status.hadErrors(ShellTest.java:81)&#10;&#9;at org.mozilla.javascript.drivers.ShellTest$3$1.run(ShellTest.java:292)&#10;&#9;at org.mozilla.javascript.Context.call(Context.java:544)&#10;&#9;at org.mozilla.javascript.ContextFactory.call(ContextFactory.java:515)&#10;&#9;at org.mozilla.javascript.drivers.ShellTest$3.run(ShellTest.java:279)&#10;&#9;at java.lang.Thread.run(Thread.java:748)&#10;&#10;&#9;at junit.framework.Assert.fail(Assert.java:57)&#10;&#9;at org.mozilla.javascript.tests.MozillaSuiteTest$JunitStatus.threw(MozillaSuiteTest.java:175)&#10;&#9;at org.mozilla.javascript.drivers.ShellTest$3$1.run(ShellTest.java:295)&#10;&#9;at org.mozilla.javascript.Context.call(Context.java:544)&#10;&#9;at org.mozilla.javascript.ContextFactory.call(ContextFactory.java:515)&#10;&#9;at org.mozilla.javascript.drivers.ShellTest$3.run(ShellTest.java:279)&#10;&#9;at java.lang.Thread.run(Thread.java:748)&#10;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: junit.framework.AssertionFailedError: junit.framework.AssertionFailedError: In &quot;testsrc/tests/Rhino_ECMA262_Tests/test_array_prototype_includes.js&quot;:
JavaScript errors in testsrc/tests/Rhino_ECMA262_Tests/test_array_prototype_includes.js:
uncaught JavaScript runtime exception: TypeError: Cannot find function includes in object /\r\n|\r|\n/.
	at junit.framework.Assert.fail(Assert.java:57)
	at org.mozilla.javascript.tests.MozillaSuiteTest$JunitStatus.failed(MozillaSuiteTest.java:159)
	at org.mozilla.javascript.drivers.ShellTest$Status.hadErrors(ShellTest.java:81)
	at org.mozilla.javascript.drivers.ShellTest$3$1.run(ShellTest.java:292)
	at org.mozilla.javascript.Context.call(Context.java:544)
	at org.mozilla.javascript.ContextFactory.call(ContextFactory.java:515)
	at org.mozilla.javascript.drivers.ShellTest$3.run(ShellTest.java:279)
	at java.lang.Thread.run(Thread.java:748)

	at junit.framework.Assert.fail(Assert.java:57)
	at org.mozilla.javascript.tests.MozillaSuiteTest$JunitStatus.threw(MozillaSuiteTest.java:175)
	at org.mozilla.javascript.drivers.ShellTest$3$1.run(ShellTest.java:295)
	at org.mozilla.javascript.Context.call(Context.java:544)
	at org.mozilla.javascript.ContextFactory.call(ContextFactory.java:515)
	at org.mozilla.javascript.drivers.ShellTest$3.run(ShellTest.java:279)
	at java.lang.Thread.run(Thread.java:748)

	at junit.framework.Assert.fail(Assert.java:57)
	at org.mozilla.javascript.tests.MozillaSuiteTest$JunitStatus.threw(MozillaSuiteTest.java:175)
	at org.mozilla.javascript.drivers.ShellTest.run(ShellTest.java:353)
	at org.mozilla.javascript.tests.MozillaSuiteTest.runMozillaTest(MozillaSuiteTest.java:192)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.junit.runners.Suite.runChild(Suite.java:128)
	at org.junit.runners.Suite.runChild(Suite.java:27)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecuter.runTestClass(JUnitTestClassExecuter.java:114)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecuter.execute(JUnitTestClassExecuter.java:57)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassProcessor.processTestClass(JUnitTestClassProcessor.java:66)
	at org.gradle.api.internal.tasks.testing.SuiteTestClassProcessor.processTestClass(SuiteTestClassProcessor.java:51)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:35)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)
	at org.gradle.internal.dispatch.ContextClassLoaderDispatch.dispatch(ContextClassLoaderDispatch.java:32)
	at org.gradle.internal.dispatch.ProxyDispatchAdapter$DispatchingInvocationHandler.invoke(ProxyDispatchAdapter.java:93)
	at com.sun.proxy.$Proxy1.processTestClass(Unknown Source)
	at org.gradle.api.internal.tasks.testing.worker.TestWorker.processTestClass(TestWorker.java:108)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:35)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)
	at org.gradle.internal.remote.internal.hub.MessageHubBackedObjectConnection$DispatchWrapper.dispatch(MessageHubBackedObjectConnection.java:146)
	at org.gradle.internal.remote.internal.hub.MessageHubBackedObjectConnection$DispatchWrapper.dispatch(MessageHubBackedObjectConnection.java:128)
	at org.gradle.internal.remote.internal.hub.MessageHub$Handler.run(MessageHub.java:404)
	at org.gradle.internal.concurrent.ExecutorPolicy$CatchAndRecordFailures.onExecute(ExecutorPolicy.java:63)
	at org.gradle.internal.concurrent.ManagedExecutorImpl$1.run(ManagedExecutorImpl.java:46)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.gradle.internal.concurrent.ThreadFactoryImpl$ManagedThreadRunnable.run(ThreadFactoryImpl.java:55)
	at java.lang.Thread.run(Thread.java:748)
</failure>
  </testcase>
  <testcase name="runMozillaTest[8, js=testsrc/tests/Rhino_ECMA262_Tests/test_array_prototype_indexof.js, opt=0]" classname="org.mozilla.javascript.tests.MozillaSuiteTest" time="8.951"/>
  <testcase name="runMozillaTest[9, js=testsrc/tests/Rhino_ECMA262_Tests/test_array_prototype_lastindexof.js, opt=0]" classname="org.mozilla.javascript.tests.MozillaSuiteTest" time="6.768"/>
  <testcase name="runMozillaTest[10, js=testsrc/tests/Rhino_ECMA262_Tests/test_array_prototype_map.js, opt=0]" classname="org.mozilla.javascript.tests.MozillaSuiteTest" time="4.418"/>
  <testcase name="runMozillaTest[11, js=testsrc/tests/Rhino_ECMA262_Tests/test_array_prototype_reduce.js, opt=0]" classname="org.mozilla.javascript.tests.MozillaSuiteTest" time="4.342"/>
  <testcase name="runMozillaTest[12, js=testsrc/tests/Rhino_ECMA262_Tests/test_array_prototype_reduceright.js, opt=0]" classname="org.mozilla.javascript.tests.MozillaSuiteTest" time="4.155"/>
  <testcase name="runMozillaTest[13, js=testsrc/tests/Rhino_ECMA262_Tests/test_array_prototype_some.js, opt=0]" classname="org.mozilla.javascript.tests.MozillaSuiteTest" time="4.192"/>
  <testcase name="runMozillaTest[14, js=testsrc/tests/Rhino_ECMA262_Tests/test_arraybuffer_isview.js, opt=0]" classname="org.mozilla.javascript.tests.MozillaSuiteTest" time="4.619"/>
  <testcase name="runMozillaTest[15, js=testsrc/tests/Rhino_ECMA262_Tests/test_arraybuffer_prototype_slice.js, opt=0]" classname="org.mozilla.javascript.tests.MozillaSuiteTest" time="4.011"/>
  <testcase name="runMozillaTest[16, js=testsrc/tests/Rhino_ECMA262_Tests/test_date_prototype_getdate.js, opt=0]" classname="org.mozilla.javascript.tests.MozillaSuiteTest" time="0.364"/>
  <testcase name="runMozillaTest[17, js=testsrc/tests/Rhino_ECMA262_Tests/test_date_prototype_getday.js, opt=0]" classname="org.mozilla.javascript.tests.MozillaSuiteTest" time="0.192"/>
  <testcase name="runMozillaTest[18, js=testsrc/tests/Rhino_ECMA262_Tests/test_date_prototype_getfullyear.js, opt=0]" classname="org.mozilla.javascript.tests.MozillaSuiteTest" time="0.185"/>
  <testcase name="runMozillaTest[19, js=testsrc/tests/Rhino_ECMA262_Tests/test_date_prototype_gethours.js, opt=0]" classname="org.mozilla.javascript.tests.MozillaSuiteTest" time="0.186"/>
  <testcase name="runMozillaTest[20, js=testsrc/tests/Rhino_ECMA262_Tests/test_date_prototype_getmilliseconds.js, opt=0]" classname="org.mozilla.javascript.tests.MozillaSuiteTest" time="0.208"/>
  <testcase name="runMozillaTest[21, js=testsrc/tests/Rhino_ECMA262_Tests/test_date_prototype_getminutes.js, opt=0]" classname="org.mozilla.javascript.tests.MozillaSuiteTest" time="0.207"/>
  <testcase name="runMozillaTest[22, js=testsrc/tests/Rhino_ECMA262_Tests/test_date_prototype_getmonth.js, opt=0]" classname="org.mozilla.javascript.tests.MozillaSuiteTest" time="0.2"/>
  <testcase name="runMozillaTest[23, js=testsrc/tests/Rhino_ECMA262_Tests/test_date_prototype_getseconds.js, opt=0]" classname="org.mozilla.javascript.tests.MozillaSuiteTest" time="0.143"/>
  <testcase name="runMozillaTest[24, js=testsrc/tests/Rhino_ECMA262_Tests/test_date_prototype_gettimezoneoffset.js, opt=0]" classname="org.mozilla.javascript.tests.MozillaSuiteTest" time="0.241"/>
  <testcase name="runMozillaTest[25, js=testsrc/tests/Rhino_ECMA262_Tests/test_date_prototype_getutcdate.js, opt=0]" classname="org.mozilla.javascript.tests.MozillaSuiteTest" time="0.226"/>
  <testcase name="runMozillaTest[26, js=testsrc/tests/Rhino_ECMA262_Tests/test_date_prototype_getutcday.js, opt=0]" classname="org.mozilla.javascript.tests.MozillaSuiteTest" time="0.27"/>
  <testcase name="runMozillaTest[27, js=testsrc/tests/Rhino_ECMA262_Tests/test_date_prototype_getutcfullyear.js, opt=0]" classname="org.mozilla.javascript.tests.MozillaSuiteTest" time="0.152"/>
  <testcase name="runMozillaTest[28, js=testsrc/tests/Rhino_ECMA262_Tests/test_date_prototype_getutchours.js, opt=0]" classname="org.mozilla.javascript.tests.MozillaSuiteTest" time="0.165"/>
  <testcase name="runMozillaTest[29, js=testsrc/tests/Rhino_ECMA262_Tests/test_date_prototype_getutcmilliseconds.js, opt=0]" classname="org.mozilla.javascript.tests.MozillaSuiteTest" time="0.221"/>
  <testcase name="runMozillaTest[30, js=testsrc/tests/Rhino_ECMA262_Tests/test_date_prototype_getutcminutes.js, opt=0]" classname="org.mozilla.javascript.tests.MozillaSuiteTest" time="0.17"/>
  <testcase name="runMozillaTest[31, js=testsrc/tests/Rhino_ECMA262_Tests/test_date_prototype_getutcmonth.js, opt=0]" classname="org.mozilla.javascript.tests.MozillaSuiteTest" time="0.174"/>
  <testcase name="runMozillaTest[32, js=testsrc/tests/Rhino_ECMA262_Tests/test_date_prototype_getutcseconds.js, opt=0]" classname="org.mozilla.javascript.tests.MozillaSuiteTest" time="0.22"/>
  <testcase name="runMozillaTest[33, js=testsrc/tests/Rhino_ECMA262_Tests/test_get_arraybuffer_prototype_bytelength.js, opt=0]" classname="org.mozilla.javascript.tests.MozillaSuiteTest" time="0.241"/>
  <testcase name="runMozillaTest[34, js=testsrc/tests/Rhino_ECMA262_Tests/test_map_prototype_clear.js, opt=0]" classname="org.mozilla.javascript.tests.MozillaSuiteTest" time="0.164">
    <failure message="junit.framework.AssertionFailedError: junit.framework.AssertionFailedError: junit.framework.AssertionFailedError: In &quot;testsrc/tests/Rhino_ECMA262_Tests/test_map_prototype_clear.js&quot;:&#10;JavaScript errors in testsrc/tests/Rhino_ECMA262_Tests/test_map_prototype_clear.js:&#10;uncaught JavaScript runtime exception: ReferenceError: &quot;Map&quot; is not defined.&#10;&#9;at junit.framework.Assert.fail(Assert.java:57)&#10;&#9;at org.mozilla.javascript.tests.MozillaSuiteTest$JunitStatus.failed(MozillaSuiteTest.java:159)&#10;&#9;at org.mozilla.javascript.drivers.ShellTest$Status.hadErrors(ShellTest.java:81)&#10;&#9;at org.mozilla.javascript.drivers.ShellTest$3$1.run(ShellTest.java:292)&#10;&#9;at org.mozilla.javascript.Context.call(Context.java:544)&#10;&#9;at org.mozilla.javascript.ContextFactory.call(ContextFactory.java:515)&#10;&#9;at org.mozilla.javascript.drivers.ShellTest$3.run(ShellTest.java:279)&#10;&#9;at java.lang.Thread.run(Thread.java:748)&#10;&#10;&#9;at junit.framework.Assert.fail(Assert.java:57)&#10;&#9;at org.mozilla.javascript.tests.MozillaSuiteTest$JunitStatus.threw(MozillaSuiteTest.java:175)&#10;&#9;at org.mozilla.javascript.drivers.ShellTest$3$1.run(ShellTest.java:295)&#10;&#9;at org.mozilla.javascript.Context.call(Context.java:544)&#10;&#9;at org.mozilla.javascript.ContextFactory.call(ContextFactory.java:515)&#10;&#9;at org.mozilla.javascript.drivers.ShellTest$3.run(ShellTest.java:279)&#10;&#9;at java.lang.Thread.run(Thread.java:748)&#10;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: junit.framework.AssertionFailedError: junit.framework.AssertionFailedError: In &quot;testsrc/tests/Rhino_ECMA262_Tests/test_map_prototype_clear.js&quot;:
JavaScript errors in testsrc/tests/Rhino_ECMA262_Tests/test_map_prototype_clear.js:
uncaught JavaScript runtime exception: ReferenceError: &quot;Map&quot; is not defined.
	at junit.framework.Assert.fail(Assert.java:57)
	at org.mozilla.javascript.tests.MozillaSuiteTest$JunitStatus.failed(MozillaSuiteTest.java:159)
	at org.mozilla.javascript.drivers.ShellTest$Status.hadErrors(ShellTest.java:81)
	at org.mozilla.javascript.drivers.ShellTest$3$1.run(ShellTest.java:292)
	at org.mozilla.javascript.Context.call(Context.java:544)
	at org.mozilla.javascript.ContextFactory.call(ContextFactory.java:515)
	at org.mozilla.javascript.drivers.ShellTest$3.run(ShellTest.java:279)
	at java.lang.Thread.run(Thread.java:748)

	at junit.framework.Assert.fail(Assert.java:57)
	at org.mozilla.javascript.tests.MozillaSuiteTest$JunitStatus.threw(MozillaSuiteTest.java:175)
	at org.mozilla.javascript.drivers.ShellTest$3$1.run(ShellTest.java:295)
	at org.mozilla.javascript.Context.call(Context.java:544)
	at org.mozilla.javascript.ContextFactory.call(ContextFactory.java:515)
	at org.mozilla.javascript.drivers.ShellTest$3.run(ShellTest.java:279)
	at java.lang.Thread.run(Thread.java:748)

	at junit.framework.Assert.fail(Assert.java:57)
	at org.mozilla.javascript.tests.MozillaSuiteTest$JunitStatus.threw(MozillaSuiteTest.java:175)
	at org.mozilla.javascript.drivers.ShellTest.run(ShellTest.java:353)
	at org.mozilla.javascript.tests.MozillaSuiteTest.runMozillaTest(MozillaSuiteTest.java:192)
	at sun.reflect.GeneratedMethodAccessor5.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.junit.runners.Suite.runChild(Suite.java:128)
	at org.junit.runners.Suite.runChild(Suite.java:27)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecuter.runTestClass(JUnitTestClassExecuter.java:114)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecuter.execute(JUnitTestClassExecuter.java:57)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassProcessor.processTestClass(JUnitTestClassProcessor.java:66)
	at org.gradle.api.internal.tasks.testing.SuiteTestClassProcessor.processTestClass(SuiteTestClassProcessor.java:51)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:35)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)
	at org.gradle.internal.dispatch.ContextClassLoaderDispatch.dispatch(ContextClassLoaderDispatch.java:32)
	at org.gradle.internal.dispatch.ProxyDispatchAdapter$DispatchingInvocationHandler.invoke(ProxyDispatchAdapter.java:93)
	at com.sun.proxy.$Proxy1.processTestClass(Unknown Source)
	at org.gradle.api.internal.tasks.testing.worker.TestWorker.processTestClass(TestWorker.java:108)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:35)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)
	at org.gradle.internal.remote.internal.hub.MessageHubBackedObjectConnection$DispatchWrapper.dispatch(MessageHubBackedObjectConnection.java:146)
	at org.gradle.internal.remote.internal.hub.MessageHubBackedObjectConnection$DispatchWrapper.dispatch(MessageHubBackedObjectConnection.java:128)
	at org.gradle.internal.remote.internal.hub.MessageHub$Handler.run(MessageHub.java:404)
	at org.gradle.internal.concurrent.ExecutorPolicy$CatchAndRecordFailures.onExecute(ExecutorPolicy.java:63)
	at org.gradle.internal.concurrent.ManagedExecutorImpl$1.run(ManagedExecutorImpl.java:46)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.gradle.internal.concurrent.ThreadFactoryImpl$ManagedThreadRunnable.run(ThreadFactoryImpl.java:55)
	at java.lang.Thread.run(Thread.java:748)
</failure>
  </testcase>
  <testcase name="runMozillaTest[35, js=testsrc/tests/Rhino_ECMA262_Tests/test_map_prototype_delete.js, opt=0]" classname="org.mozilla.javascript.tests.MozillaSuiteTest" time="0.115">
    <failure message="junit.framework.AssertionFailedError: junit.framework.AssertionFailedError: junit.framework.AssertionFailedError: In &quot;testsrc/tests/Rhino_ECMA262_Tests/test_map_prototype_delete.js&quot;:&#10;JavaScript errors in testsrc/tests/Rhino_ECMA262_Tests/test_map_prototype_delete.js:&#10;testsrc/tests/Rhino_ECMA262_Tests/test_map_prototype_delete.js:120: missing name after . operator&#10;&#9;&#9;&#9;var output = randominput.delete(key);&#10;&#9;&#9;&#9;                               ^&#10;testsrc/tests/Rhino_ECMA262_Tests/test_map_prototype_delete.js:130: missing name after . operator&#10;&#9;&#9;&#9;var output = randominput.delete(key);&#10;&#9;&#9;&#9;                               ^&#10;testsrc/tests/Rhino_ECMA262_Tests/test_map_prototype_delete.js:1: Compilation produced 2 syntax errors.&#10;&#9;at junit.framework.Assert.fail(Assert.java:57)&#10;&#9;at org.mozilla.javascript.tests.MozillaSuiteTest$JunitStatus.failed(MozillaSuiteTest.java:159)&#10;&#9;at org.mozilla.javascript.drivers.ShellTest$Status.hadErrors(ShellTest.java:81)&#10;&#9;at org.mozilla.javascript.drivers.ShellTest$3$1.run(ShellTest.java:292)&#10;&#9;at org.mozilla.javascript.Context.call(Context.java:544)&#10;&#9;at org.mozilla.javascript.ContextFactory.call(ContextFactory.java:515)&#10;&#9;at org.mozilla.javascript.drivers.ShellTest$3.run(ShellTest.java:279)&#10;&#9;at java.lang.Thread.run(Thread.java:748)&#10;&#10;&#9;at junit.framework.Assert.fail(Assert.java:57)&#10;&#9;at org.mozilla.javascript.tests.MozillaSuiteTest$JunitStatus.threw(MozillaSuiteTest.java:175)&#10;&#9;at org.mozilla.javascript.drivers.ShellTest$3$1.run(ShellTest.java:295)&#10;&#9;at org.mozilla.javascript.Context.call(Context.java:544)&#10;&#9;at org.mozilla.javascript.ContextFactory.call(ContextFactory.java:515)&#10;&#9;at org.mozilla.javascript.drivers.ShellTest$3.run(ShellTest.java:279)&#10;&#9;at java.lang.Thread.run(Thread.java:748)&#10;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: junit.framework.AssertionFailedError: junit.framework.AssertionFailedError: In &quot;testsrc/tests/Rhino_ECMA262_Tests/test_map_prototype_delete.js&quot;:
JavaScript errors in testsrc/tests/Rhino_ECMA262_Tests/test_map_prototype_delete.js:
testsrc/tests/Rhino_ECMA262_Tests/test_map_prototype_delete.js:120: missing name after . operator
			var output = randominput.delete(key);
			                               ^
testsrc/tests/Rhino_ECMA262_Tests/test_map_prototype_delete.js:130: missing name after . operator
			var output = randominput.delete(key);
			                               ^
testsrc/tests/Rhino_ECMA262_Tests/test_map_prototype_delete.js:1: Compilation produced 2 syntax errors.
	at junit.framework.Assert.fail(Assert.java:57)
	at org.mozilla.javascript.tests.MozillaSuiteTest$JunitStatus.failed(MozillaSuiteTest.java:159)
	at org.mozilla.javascript.drivers.ShellTest$Status.hadErrors(ShellTest.java:81)
	at org.mozilla.javascript.drivers.ShellTest$3$1.run(ShellTest.java:292)
	at org.mozilla.javascript.Context.call(Context.java:544)
	at org.mozilla.javascript.ContextFactory.call(ContextFactory.java:515)
	at org.mozilla.javascript.drivers.ShellTest$3.run(ShellTest.java:279)
	at java.lang.Thread.run(Thread.java:748)

	at junit.framework.Assert.fail(Assert.java:57)
	at org.mozilla.javascript.tests.MozillaSuiteTest$JunitStatus.threw(MozillaSuiteTest.java:175)
	at org.mozilla.javascript.drivers.ShellTest$3$1.run(ShellTest.java:295)
	at org.mozilla.javascript.Context.call(Context.java:544)
	at org.mozilla.javascript.ContextFactory.call(ContextFactory.java:515)
	at org.mozilla.javascript.drivers.ShellTest$3.run(ShellTest.java:279)
	at java.lang.Thread.run(Thread.java:748)

	at junit.framework.Assert.fail(Assert.java:57)
	at org.mozilla.javascript.tests.MozillaSuiteTest$JunitStatus.threw(MozillaSuiteTest.java:175)
	at org.mozilla.javascript.drivers.ShellTest.run(ShellTest.java:353)
	at org.mozilla.javascript.tests.MozillaSuiteTest.runMozillaTest(MozillaSuiteTest.java:192)
	at sun.reflect.GeneratedMethodAccessor5.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.junit.runners.Suite.runChild(Suite.java:128)
	at org.junit.runners.Suite.runChild(Suite.java:27)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecuter.runTestClass(JUnitTestClassExecuter.java:114)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecuter.execute(JUnitTestClassExecuter.java:57)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassProcessor.processTestClass(JUnitTestClassProcessor.java:66)
	at org.gradle.api.internal.tasks.testing.SuiteTestClassProcessor.processTestClass(SuiteTestClassProcessor.java:51)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:35)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)
	at org.gradle.internal.dispatch.ContextClassLoaderDispatch.dispatch(ContextClassLoaderDispatch.java:32)
	at org.gradle.internal.dispatch.ProxyDispatchAdapter$DispatchingInvocationHandler.invoke(ProxyDispatchAdapter.java:93)
	at com.sun.proxy.$Proxy1.processTestClass(Unknown Source)
	at org.gradle.api.internal.tasks.testing.worker.TestWorker.processTestClass(TestWorker.java:108)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:35)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)
	at org.gradle.internal.remote.internal.hub.MessageHubBackedObjectConnection$DispatchWrapper.dispatch(MessageHubBackedObjectConnection.java:146)
	at org.gradle.internal.remote.internal.hub.MessageHubBackedObjectConnection$DispatchWrapper.dispatch(MessageHubBackedObjectConnection.java:128)
	at org.gradle.internal.remote.internal.hub.MessageHub$Handler.run(MessageHub.java:404)
	at org.gradle.internal.concurrent.ExecutorPolicy$CatchAndRecordFailures.onExecute(ExecutorPolicy.java:63)
	at org.gradle.internal.concurrent.ManagedExecutorImpl$1.run(ManagedExecutorImpl.java:46)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.gradle.internal.concurrent.ThreadFactoryImpl$ManagedThreadRunnable.run(ThreadFactoryImpl.java:55)
	at java.lang.Thread.run(Thread.java:748)
</failure>
  </testcase>
  <testcase name="runMozillaTest[36, js=testsrc/tests/Rhino_ECMA262_Tests/test_map_prototype_foreach.js, opt=0]" classname="org.mozilla.javascript.tests.MozillaSuiteTest" time="0.421">
    <failure message="junit.framework.AssertionFailedError: junit.framework.AssertionFailedError: junit.framework.AssertionFailedError: In &quot;testsrc/tests/Rhino_ECMA262_Tests/test_map_prototype_foreach.js&quot;:&#10;JavaScript errors in testsrc/tests/Rhino_ECMA262_Tests/test_map_prototype_foreach.js:&#10;uncaught JavaScript runtime exception: ReferenceError: &quot;Map&quot; is not defined.&#10;&#9;at junit.framework.Assert.fail(Assert.java:57)&#10;&#9;at org.mozilla.javascript.tests.MozillaSuiteTest$JunitStatus.failed(MozillaSuiteTest.java:159)&#10;&#9;at org.mozilla.javascript.drivers.ShellTest$Status.hadErrors(ShellTest.java:81)&#10;&#9;at org.mozilla.javascript.drivers.ShellTest$3$1.run(ShellTest.java:292)&#10;&#9;at org.mozilla.javascript.Context.call(Context.java:544)&#10;&#9;at org.mozilla.javascript.ContextFactory.call(ContextFactory.java:515)&#10;&#9;at org.mozilla.javascript.drivers.ShellTest$3.run(ShellTest.java:279)&#10;&#9;at java.lang.Thread.run(Thread.java:748)&#10;&#10;&#9;at junit.framework.Assert.fail(Assert.java:57)&#10;&#9;at org.mozilla.javascript.tests.MozillaSuiteTest$JunitStatus.threw(MozillaSuiteTest.java:175)&#10;&#9;at org.mozilla.javascript.drivers.ShellTest$3$1.run(ShellTest.java:295)&#10;&#9;at org.mozilla.javascript.Context.call(Context.java:544)&#10;&#9;at org.mozilla.javascript.ContextFactory.call(ContextFactory.java:515)&#10;&#9;at org.mozilla.javascript.drivers.ShellTest$3.run(ShellTest.java:279)&#10;&#9;at java.lang.Thread.run(Thread.java:748)&#10;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: junit.framework.AssertionFailedError: junit.framework.AssertionFailedError: In &quot;testsrc/tests/Rhino_ECMA262_Tests/test_map_prototype_foreach.js&quot;:
JavaScript errors in testsrc/tests/Rhino_ECMA262_Tests/test_map_prototype_foreach.js:
uncaught JavaScript runtime exception: ReferenceError: &quot;Map&quot; is not defined.
	at junit.framework.Assert.fail(Assert.java:57)
	at org.mozilla.javascript.tests.MozillaSuiteTest$JunitStatus.failed(MozillaSuiteTest.java:159)
	at org.mozilla.javascript.drivers.ShellTest$Status.hadErrors(ShellTest.java:81)
	at org.mozilla.javascript.drivers.ShellTest$3$1.run(ShellTest.java:292)
	at org.mozilla.javascript.Context.call(Context.java:544)
	at org.mozilla.javascript.ContextFactory.call(ContextFactory.java:515)
	at org.mozilla.javascript.drivers.ShellTest$3.run(ShellTest.java:279)
	at java.lang.Thread.run(Thread.java:748)

	at junit.framework.Assert.fail(Assert.java:57)
	at org.mozilla.javascript.tests.MozillaSuiteTest$JunitStatus.threw(MozillaSuiteTest.java:175)
	at org.mozilla.javascript.drivers.ShellTest$3$1.run(ShellTest.java:295)
	at org.mozilla.javascript.Context.call(Context.java:544)
	at org.mozilla.javascript.ContextFactory.call(ContextFactory.java:515)
	at org.mozilla.javascript.drivers.ShellTest$3.run(ShellTest.java:279)
	at java.lang.Thread.run(Thread.java:748)

	at junit.framework.Assert.fail(Assert.java:57)
	at org.mozilla.javascript.tests.MozillaSuiteTest$JunitStatus.threw(MozillaSuiteTest.java:175)
	at org.mozilla.javascript.drivers.ShellTest.run(ShellTest.java:353)
	at org.mozilla.javascript.tests.MozillaSuiteTest.runMozillaTest(MozillaSuiteTest.java:192)
	at sun.reflect.GeneratedMethodAccessor5.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.junit.runners.Suite.runChild(Suite.java:128)
	at org.junit.runners.Suite.runChild(Suite.java:27)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecuter.runTestClass(JUnitTestClassExecuter.java:114)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecuter.execute(JUnitTestClassExecuter.java:57)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassProcessor.processTestClass(JUnitTestClassProcessor.java:66)
	at org.gradle.api.internal.tasks.testing.SuiteTestClassProcessor.processTestClass(SuiteTestClassProcessor.java:51)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:35)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)
	at org.gradle.internal.dispatch.ContextClassLoaderDispatch.dispatch(ContextClassLoaderDispatch.java:32)
	at org.gradle.internal.dispatch.ProxyDispatchAdapter$DispatchingInvocationHandler.invoke(ProxyDispatchAdapter.java:93)
	at com.sun.proxy.$Proxy1.processTestClass(Unknown Source)
	at org.gradle.api.internal.tasks.testing.worker.TestWorker.processTestClass(TestWorker.java:108)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:35)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)
	at org.gradle.internal.remote.internal.hub.MessageHubBackedObjectConnection$DispatchWrapper.dispatch(MessageHubBackedObjectConnection.java:146)
	at org.gradle.internal.remote.internal.hub.MessageHubBackedObjectConnection$DispatchWrapper.dispatch(MessageHubBackedObjectConnection.java:128)
	at org.gradle.internal.remote.internal.hub.MessageHub$Handler.run(MessageHub.java:404)
	at org.gradle.internal.concurrent.ExecutorPolicy$CatchAndRecordFailures.onExecute(ExecutorPolicy.java:63)
	at org.gradle.internal.concurrent.ManagedExecutorImpl$1.run(ManagedExecutorImpl.java:46)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.gradle.internal.concurrent.ThreadFactoryImpl$ManagedThreadRunnable.run(ThreadFactoryImpl.java:55)
	at java.lang.Thread.run(Thread.java:748)
</failure>
  </testcase>
  <testcase name="runMozillaTest[37, js=testsrc/tests/Rhino_ECMA262_Tests/test_map_prototype_get.js, opt=0]" classname="org.mozilla.javascript.tests.MozillaSuiteTest" time="0.374">
    <failure message="junit.framework.AssertionFailedError: junit.framework.AssertionFailedError: junit.framework.AssertionFailedError: In &quot;testsrc/tests/Rhino_ECMA262_Tests/test_map_prototype_get.js&quot;:&#10;JavaScript errors in testsrc/tests/Rhino_ECMA262_Tests/test_map_prototype_get.js:&#10;uncaught JavaScript runtime exception: ReferenceError: &quot;Map&quot; is not defined.&#10;&#9;at junit.framework.Assert.fail(Assert.java:57)&#10;&#9;at org.mozilla.javascript.tests.MozillaSuiteTest$JunitStatus.failed(MozillaSuiteTest.java:159)&#10;&#9;at org.mozilla.javascript.drivers.ShellTest$Status.hadErrors(ShellTest.java:81)&#10;&#9;at org.mozilla.javascript.drivers.ShellTest$3$1.run(ShellTest.java:292)&#10;&#9;at org.mozilla.javascript.Context.call(Context.java:544)&#10;&#9;at org.mozilla.javascript.ContextFactory.call(ContextFactory.java:515)&#10;&#9;at org.mozilla.javascript.drivers.ShellTest$3.run(ShellTest.java:279)&#10;&#9;at java.lang.Thread.run(Thread.java:748)&#10;&#10;&#9;at junit.framework.Assert.fail(Assert.java:57)&#10;&#9;at org.mozilla.javascript.tests.MozillaSuiteTest$JunitStatus.threw(MozillaSuiteTest.java:175)&#10;&#9;at org.mozilla.javascript.drivers.ShellTest$3$1.run(ShellTest.java:295)&#10;&#9;at org.mozilla.javascript.Context.call(Context.java:544)&#10;&#9;at org.mozilla.javascript.ContextFactory.call(ContextFactory.java:515)&#10;&#9;at org.mozilla.javascript.drivers.ShellTest$3.run(ShellTest.java:279)&#10;&#9;at java.lang.Thread.run(Thread.java:748)&#10;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: junit.framework.AssertionFailedError: junit.framework.AssertionFailedError: In &quot;testsrc/tests/Rhino_ECMA262_Tests/test_map_prototype_get.js&quot;:
JavaScript errors in testsrc/tests/Rhino_ECMA262_Tests/test_map_prototype_get.js:
uncaught JavaScript runtime exception: ReferenceError: &quot;Map&quot; is not defined.
	at junit.framework.Assert.fail(Assert.java:57)
	at org.mozilla.javascript.tests.MozillaSuiteTest$JunitStatus.failed(MozillaSuiteTest.java:159)
	at org.mozilla.javascript.drivers.ShellTest$Status.hadErrors(ShellTest.java:81)
	at org.mozilla.javascript.drivers.ShellTest$3$1.run(ShellTest.java:292)
	at org.mozilla.javascript.Context.call(Context.java:544)
	at org.mozilla.javascript.ContextFactory.call(ContextFactory.java:515)
	at org.mozilla.javascript.drivers.ShellTest$3.run(ShellTest.java:279)
	at java.lang.Thread.run(Thread.java:748)

	at junit.framework.Assert.fail(Assert.java:57)
	at org.mozilla.javascript.tests.MozillaSuiteTest$JunitStatus.threw(MozillaSuiteTest.java:175)
	at org.mozilla.javascript.drivers.ShellTest$3$1.run(ShellTest.java:295)
	at org.mozilla.javascript.Context.call(Context.java:544)
	at org.mozilla.javascript.ContextFactory.call(ContextFactory.java:515)
	at org.mozilla.javascript.drivers.ShellTest$3.run(ShellTest.java:279)
	at java.lang.Thread.run(Thread.java:748)

	at junit.framework.Assert.fail(Assert.java:57)
	at org.mozilla.javascript.tests.MozillaSuiteTest$JunitStatus.threw(MozillaSuiteTest.java:175)
	at org.mozilla.javascript.drivers.ShellTest.run(ShellTest.java:353)
	at org.mozilla.javascript.tests.MozillaSuiteTest.runMozillaTest(MozillaSuiteTest.java:192)
	at sun.reflect.GeneratedMethodAccessor5.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.junit.runners.Suite.runChild(Suite.java:128)
	at org.junit.runners.Suite.runChild(Suite.java:27)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecuter.runTestClass(JUnitTestClassExecuter.java:114)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecuter.execute(JUnitTestClassExecuter.java:57)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassProcessor.processTestClass(JUnitTestClassProcessor.java:66)
	at org.gradle.api.internal.tasks.testing.SuiteTestClassProcessor.processTestClass(SuiteTestClassProcessor.java:51)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:35)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)
	at org.gradle.internal.dispatch.ContextClassLoaderDispatch.dispatch(ContextClassLoaderDispatch.java:32)
	at org.gradle.internal.dispatch.ProxyDispatchAdapter$DispatchingInvocationHandler.invoke(ProxyDispatchAdapter.java:93)
	at com.sun.proxy.$Proxy1.processTestClass(Unknown Source)
	at org.gradle.api.internal.tasks.testing.worker.TestWorker.processTestClass(TestWorker.java:108)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:35)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)
	at org.gradle.internal.remote.internal.hub.MessageHubBackedObjectConnection$DispatchWrapper.dispatch(MessageHubBackedObjectConnection.java:146)
	at org.gradle.internal.remote.internal.hub.MessageHubBackedObjectConnection$DispatchWrapper.dispatch(MessageHubBackedObjectConnection.java:128)
	at org.gradle.internal.remote.internal.hub.MessageHub$Handler.run(MessageHub.java:404)
	at org.gradle.internal.concurrent.ExecutorPolicy$CatchAndRecordFailures.onExecute(ExecutorPolicy.java:63)
	at org.gradle.internal.concurrent.ManagedExecutorImpl$1.run(ManagedExecutorImpl.java:46)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.gradle.internal.concurrent.ThreadFactoryImpl$ManagedThreadRunnable.run(ThreadFactoryImpl.java:55)
	at java.lang.Thread.run(Thread.java:748)
</failure>
  </testcase>
  <testcase name="runMozillaTest[38, js=testsrc/tests/Rhino_ECMA262_Tests/test_map_prototype_has.js, opt=0]" classname="org.mozilla.javascript.tests.MozillaSuiteTest" time="0.391">
    <failure message="junit.framework.AssertionFailedError: junit.framework.AssertionFailedError: junit.framework.AssertionFailedError: In &quot;testsrc/tests/Rhino_ECMA262_Tests/test_map_prototype_has.js&quot;:&#10;JavaScript errors in testsrc/tests/Rhino_ECMA262_Tests/test_map_prototype_has.js:&#10;uncaught JavaScript runtime exception: ReferenceError: &quot;Map&quot; is not defined.&#10;&#9;at junit.framework.Assert.fail(Assert.java:57)&#10;&#9;at org.mozilla.javascript.tests.MozillaSuiteTest$JunitStatus.failed(MozillaSuiteTest.java:159)&#10;&#9;at org.mozilla.javascript.drivers.ShellTest$Status.hadErrors(ShellTest.java:81)&#10;&#9;at org.mozilla.javascript.drivers.ShellTest$3$1.run(ShellTest.java:292)&#10;&#9;at org.mozilla.javascript.Context.call(Context.java:544)&#10;&#9;at org.mozilla.javascript.ContextFactory.call(ContextFactory.java:515)&#10;&#9;at org.mozilla.javascript.drivers.ShellTest$3.run(ShellTest.java:279)&#10;&#9;at java.lang.Thread.run(Thread.java:748)&#10;&#10;&#9;at junit.framework.Assert.fail(Assert.java:57)&#10;&#9;at org.mozilla.javascript.tests.MozillaSuiteTest$JunitStatus.threw(MozillaSuiteTest.java:175)&#10;&#9;at org.mozilla.javascript.drivers.ShellTest$3$1.run(ShellTest.java:295)&#10;&#9;at org.mozilla.javascript.Context.call(Context.java:544)&#10;&#9;at org.mozilla.javascript.ContextFactory.call(ContextFactory.java:515)&#10;&#9;at org.mozilla.javascript.drivers.ShellTest$3.run(ShellTest.java:279)&#10;&#9;at java.lang.Thread.run(Thread.java:748)&#10;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: junit.framework.AssertionFailedError: junit.framework.AssertionFailedError: In &quot;testsrc/tests/Rhino_ECMA262_Tests/test_map_prototype_has.js&quot;:
JavaScript errors in testsrc/tests/Rhino_ECMA262_Tests/test_map_prototype_has.js:
uncaught JavaScript runtime exception: ReferenceError: &quot;Map&quot; is not defined.
	at junit.framework.Assert.fail(Assert.java:57)
	at org.mozilla.javascript.tests.MozillaSuiteTest$JunitStatus.failed(MozillaSuiteTest.java:159)
	at org.mozilla.javascript.drivers.ShellTest$Status.hadErrors(ShellTest.java:81)
	at org.mozilla.javascript.drivers.ShellTest$3$1.run(ShellTest.java:292)
	at org.mozilla.javascript.Context.call(Context.java:544)
	at org.mozilla.javascript.ContextFactory.call(ContextFactory.java:515)
	at org.mozilla.javascript.drivers.ShellTest$3.run(ShellTest.java:279)
	at java.lang.Thread.run(Thread.java:748)

	at junit.framework.Assert.fail(Assert.java:57)
	at org.mozilla.javascript.tests.MozillaSuiteTest$JunitStatus.threw(MozillaSuiteTest.java:175)
	at org.mozilla.javascript.drivers.ShellTest$3$1.run(ShellTest.java:295)
	at org.mozilla.javascript.Context.call(Context.java:544)
	at org.mozilla.javascript.ContextFactory.call(ContextFactory.java:515)
	at org.mozilla.javascript.drivers.ShellTest$3.run(ShellTest.java:279)
	at java.lang.Thread.run(Thread.java:748)

	at junit.framework.Assert.fail(Assert.java:57)
	at org.mozilla.javascript.tests.MozillaSuiteTest$JunitStatus.threw(MozillaSuiteTest.java:175)
	at org.mozilla.javascript.drivers.ShellTest.run(ShellTest.java:353)
	at org.mozilla.javascript.tests.MozillaSuiteTest.runMozillaTest(MozillaSuiteTest.java:192)
	at sun.reflect.GeneratedMethodAccessor5.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.junit.runners.Suite.runChild(Suite.java:128)
	at org.junit.runners.Suite.runChild(Suite.java:27)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecuter.runTestClass(JUnitTestClassExecuter.java:114)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecuter.execute(JUnitTestClassExecuter.java:57)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassProcessor.processTestClass(JUnitTestClassProcessor.java:66)
	at org.gradle.api.internal.tasks.testing.SuiteTestClassProcessor.processTestClass(SuiteTestClassProcessor.java:51)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:35)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)
	at org.gradle.internal.dispatch.ContextClassLoaderDispatch.dispatch(ContextClassLoaderDispatch.java:32)
	at org.gradle.internal.dispatch.ProxyDispatchAdapter$DispatchingInvocationHandler.invoke(ProxyDispatchAdapter.java:93)
	at com.sun.proxy.$Proxy1.processTestClass(Unknown Source)
	at org.gradle.api.internal.tasks.testing.worker.TestWorker.processTestClass(TestWorker.java:108)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:35)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)
	at org.gradle.internal.remote.internal.hub.MessageHubBackedObjectConnection$DispatchWrapper.dispatch(MessageHubBackedObjectConnection.java:146)
	at org.gradle.internal.remote.internal.hub.MessageHubBackedObjectConnection$DispatchWrapper.dispatch(MessageHubBackedObjectConnection.java:128)
	at org.gradle.internal.remote.internal.hub.MessageHub$Handler.run(MessageHub.java:404)
	at org.gradle.internal.concurrent.ExecutorPolicy$CatchAndRecordFailures.onExecute(ExecutorPolicy.java:63)
	at org.gradle.internal.concurrent.ManagedExecutorImpl$1.run(ManagedExecutorImpl.java:46)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.gradle.internal.concurrent.ThreadFactoryImpl$ManagedThreadRunnable.run(ThreadFactoryImpl.java:55)
	at java.lang.Thread.run(Thread.java:748)
</failure>
  </testcase>
  <testcase name="runMozillaTest[39, js=testsrc/tests/Rhino_ECMA262_Tests/test_map_prototype_set.js, opt=0]" classname="org.mozilla.javascript.tests.MozillaSuiteTest" time="0.537">
    <failure message="junit.framework.AssertionFailedError: junit.framework.AssertionFailedError: junit.framework.AssertionFailedError: In &quot;testsrc/tests/Rhino_ECMA262_Tests/test_map_prototype_set.js&quot;:&#10;JavaScript errors in testsrc/tests/Rhino_ECMA262_Tests/test_map_prototype_set.js:&#10;uncaught JavaScript runtime exception: ReferenceError: &quot;Map&quot; is not defined.&#10;&#9;at junit.framework.Assert.fail(Assert.java:57)&#10;&#9;at org.mozilla.javascript.tests.MozillaSuiteTest$JunitStatus.failed(MozillaSuiteTest.java:159)&#10;&#9;at org.mozilla.javascript.drivers.ShellTest$Status.hadErrors(ShellTest.java:81)&#10;&#9;at org.mozilla.javascript.drivers.ShellTest$3$1.run(ShellTest.java:292)&#10;&#9;at org.mozilla.javascript.Context.call(Context.java:544)&#10;&#9;at org.mozilla.javascript.ContextFactory.call(ContextFactory.java:515)&#10;&#9;at org.mozilla.javascript.drivers.ShellTest$3.run(ShellTest.java:279)&#10;&#9;at java.lang.Thread.run(Thread.java:748)&#10;&#10;&#9;at junit.framework.Assert.fail(Assert.java:57)&#10;&#9;at org.mozilla.javascript.tests.MozillaSuiteTest$JunitStatus.threw(MozillaSuiteTest.java:175)&#10;&#9;at org.mozilla.javascript.drivers.ShellTest$3$1.run(ShellTest.java:295)&#10;&#9;at org.mozilla.javascript.Context.call(Context.java:544)&#10;&#9;at org.mozilla.javascript.ContextFactory.call(ContextFactory.java:515)&#10;&#9;at org.mozilla.javascript.drivers.ShellTest$3.run(ShellTest.java:279)&#10;&#9;at java.lang.Thread.run(Thread.java:748)&#10;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: junit.framework.AssertionFailedError: junit.framework.AssertionFailedError: In &quot;testsrc/tests/Rhino_ECMA262_Tests/test_map_prototype_set.js&quot;:
JavaScript errors in testsrc/tests/Rhino_ECMA262_Tests/test_map_prototype_set.js:
uncaught JavaScript runtime exception: ReferenceError: &quot;Map&quot; is not defined.
	at junit.framework.Assert.fail(Assert.java:57)
	at org.mozilla.javascript.tests.MozillaSuiteTest$JunitStatus.failed(MozillaSuiteTest.java:159)
	at org.mozilla.javascript.drivers.ShellTest$Status.hadErrors(ShellTest.java:81)
	at org.mozilla.javascript.drivers.ShellTest$3$1.run(ShellTest.java:292)
	at org.mozilla.javascript.Context.call(Context.java:544)
	at org.mozilla.javascript.ContextFactory.call(ContextFactory.java:515)
	at org.mozilla.javascript.drivers.ShellTest$3.run(ShellTest.java:279)
	at java.lang.Thread.run(Thread.java:748)

	at junit.framework.Assert.fail(Assert.java:57)
	at org.mozilla.javascript.tests.MozillaSuiteTest$JunitStatus.threw(MozillaSuiteTest.java:175)
	at org.mozilla.javascript.drivers.ShellTest$3$1.run(ShellTest.java:295)
	at org.mozilla.javascript.Context.call(Context.java:544)
	at org.mozilla.javascript.ContextFactory.call(ContextFactory.java:515)
	at org.mozilla.javascript.drivers.ShellTest$3.run(ShellTest.java:279)
	at java.lang.Thread.run(Thread.java:748)

	at junit.framework.Assert.fail(Assert.java:57)
	at org.mozilla.javascript.tests.MozillaSuiteTest$JunitStatus.threw(MozillaSuiteTest.java:175)
	at org.mozilla.javascript.drivers.ShellTest.run(ShellTest.java:353)
	at org.mozilla.javascript.tests.MozillaSuiteTest.runMozillaTest(MozillaSuiteTest.java:192)
	at sun.reflect.GeneratedMethodAccessor5.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.junit.runners.Suite.runChild(Suite.java:128)
	at org.junit.runners.Suite.runChild(Suite.java:27)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecuter.runTestClass(JUnitTestClassExecuter.java:114)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecuter.execute(JUnitTestClassExecuter.java:57)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassProcessor.processTestClass(JUnitTestClassProcessor.java:66)
	at org.gradle.api.internal.tasks.testing.SuiteTestClassProcessor.processTestClass(SuiteTestClassProcessor.java:51)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:35)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)
	at org.gradle.internal.dispatch.ContextClassLoaderDispatch.dispatch(ContextClassLoaderDispatch.java:32)
	at org.gradle.internal.dispatch.ProxyDispatchAdapter$DispatchingInvocationHandler.invoke(ProxyDispatchAdapter.java:93)
	at com.sun.proxy.$Proxy1.processTestClass(Unknown Source)
	at org.gradle.api.internal.tasks.testing.worker.TestWorker.processTestClass(TestWorker.java:108)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:35)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)
	at org.gradle.internal.remote.internal.hub.MessageHubBackedObjectConnection$DispatchWrapper.dispatch(MessageHubBackedObjectConnection.java:146)
	at org.gradle.internal.remote.internal.hub.MessageHubBackedObjectConnection$DispatchWrapper.dispatch(MessageHubBackedObjectConnection.java:128)
	at org.gradle.internal.remote.internal.hub.MessageHub$Handler.run(MessageHub.java:404)
	at org.gradle.internal.concurrent.ExecutorPolicy$CatchAndRecordFailures.onExecute(ExecutorPolicy.java:63)
	at org.gradle.internal.concurrent.ManagedExecutorImpl$1.run(ManagedExecutorImpl.java:46)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.gradle.internal.concurrent.ThreadFactoryImpl$ManagedThreadRunnable.run(ThreadFactoryImpl.java:55)
	at java.lang.Thread.run(Thread.java:748)
</failure>
  </testcase>
  <testcase name="runMozillaTest[40, js=testsrc/tests/Rhino_ECMA262_Tests/test_math_abs.js, opt=0]" classname="org.mozilla.javascript.tests.MozillaSuiteTest" time="0.228"/>
  <testcase name="runMozillaTest[41, js=testsrc/tests/Rhino_ECMA262_Tests/test_math_acos.js, opt=0]" classname="org.mozilla.javascript.tests.MozillaSuiteTest" time="2.819"/>
  <testcase name="runMozillaTest[42, js=testsrc/tests/Rhino_ECMA262_Tests/test_math_acosh.js, opt=0]" classname="org.mozilla.javascript.tests.MozillaSuiteTest" time="0.111">
    <failure message="junit.framework.AssertionFailedError: junit.framework.AssertionFailedError: junit.framework.AssertionFailedError: In &quot;testsrc/tests/Rhino_ECMA262_Tests/test_math_acosh.js&quot;:&#10;JavaScript errors in testsrc/tests/Rhino_ECMA262_Tests/test_math_acosh.js:&#10;uncaught JavaScript runtime exception: TypeError: Cannot find function acosh in object [object Math].&#10;&#9;at junit.framework.Assert.fail(Assert.java:57)&#10;&#9;at org.mozilla.javascript.tests.MozillaSuiteTest$JunitStatus.failed(MozillaSuiteTest.java:159)&#10;&#9;at org.mozilla.javascript.drivers.ShellTest$Status.hadErrors(ShellTest.java:81)&#10;&#9;at org.mozilla.javascript.drivers.ShellTest$3$1.run(ShellTest.java:292)&#10;&#9;at org.mozilla.javascript.Context.call(Context.java:544)&#10;&#9;at org.mozilla.javascript.ContextFactory.call(ContextFactory.java:515)&#10;&#9;at org.mozilla.javascript.drivers.ShellTest$3.run(ShellTest.java:279)&#10;&#9;at java.lang.Thread.run(Thread.java:748)&#10;&#10;&#9;at junit.framework.Assert.fail(Assert.java:57)&#10;&#9;at org.mozilla.javascript.tests.MozillaSuiteTest$JunitStatus.threw(MozillaSuiteTest.java:175)&#10;&#9;at org.mozilla.javascript.drivers.ShellTest$3$1.run(ShellTest.java:295)&#10;&#9;at org.mozilla.javascript.Context.call(Context.java:544)&#10;&#9;at org.mozilla.javascript.ContextFactory.call(ContextFactory.java:515)&#10;&#9;at org.mozilla.javascript.drivers.ShellTest$3.run(ShellTest.java:279)&#10;&#9;at java.lang.Thread.run(Thread.java:748)&#10;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: junit.framework.AssertionFailedError: junit.framework.AssertionFailedError: In &quot;testsrc/tests/Rhino_ECMA262_Tests/test_math_acosh.js&quot;:
JavaScript errors in testsrc/tests/Rhino_ECMA262_Tests/test_math_acosh.js:
uncaught JavaScript runtime exception: TypeError: Cannot find function acosh in object [object Math].
	at junit.framework.Assert.fail(Assert.java:57)
	at org.mozilla.javascript.tests.MozillaSuiteTest$JunitStatus.failed(MozillaSuiteTest.java:159)
	at org.mozilla.javascript.drivers.ShellTest$Status.hadErrors(ShellTest.java:81)
	at org.mozilla.javascript.drivers.ShellTest$3$1.run(ShellTest.java:292)
	at org.mozilla.javascript.Context.call(Context.java:544)
	at org.mozilla.javascript.ContextFactory.call(ContextFactory.java:515)
	at org.mozilla.javascript.drivers.ShellTest$3.run(ShellTest.java:279)
	at java.lang.Thread.run(Thread.java:748)

	at junit.framework.Assert.fail(Assert.java:57)
	at org.mozilla.javascript.tests.MozillaSuiteTest$JunitStatus.threw(MozillaSuiteTest.java:175)
	at org.mozilla.javascript.drivers.ShellTest$3$1.run(ShellTest.java:295)
	at org.mozilla.javascript.Context.call(Context.java:544)
	at org.mozilla.javascript.ContextFactory.call(ContextFactory.java:515)
	at org.mozilla.javascript.drivers.ShellTest$3.run(ShellTest.java:279)
	at java.lang.Thread.run(Thread.java:748)

	at junit.framework.Assert.fail(Assert.java:57)
	at org.mozilla.javascript.tests.MozillaSuiteTest$JunitStatus.threw(MozillaSuiteTest.java:175)
	at org.mozilla.javascript.drivers.ShellTest.run(ShellTest.java:353)
	at org.mozilla.javascript.tests.MozillaSuiteTest.runMozillaTest(MozillaSuiteTest.java:192)
	at sun.reflect.GeneratedMethodAccessor5.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.junit.runners.Suite.runChild(Suite.java:128)
	at org.junit.runners.Suite.runChild(Suite.java:27)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecuter.runTestClass(JUnitTestClassExecuter.java:114)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecuter.execute(JUnitTestClassExecuter.java:57)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassProcessor.processTestClass(JUnitTestClassProcessor.java:66)
	at org.gradle.api.internal.tasks.testing.SuiteTestClassProcessor.processTestClass(SuiteTestClassProcessor.java:51)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:35)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)
	at org.gradle.internal.dispatch.ContextClassLoaderDispatch.dispatch(ContextClassLoaderDispatch.java:32)
	at org.gradle.internal.dispatch.ProxyDispatchAdapter$DispatchingInvocationHandler.invoke(ProxyDispatchAdapter.java:93)
	at com.sun.proxy.$Proxy1.processTestClass(Unknown Source)
	at org.gradle.api.internal.tasks.testing.worker.TestWorker.processTestClass(TestWorker.java:108)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:35)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)
	at org.gradle.internal.remote.internal.hub.MessageHubBackedObjectConnection$DispatchWrapper.dispatch(MessageHubBackedObjectConnection.java:146)
	at org.gradle.internal.remote.internal.hub.MessageHubBackedObjectConnection$DispatchWrapper.dispatch(MessageHubBackedObjectConnection.java:128)
	at org.gradle.internal.remote.internal.hub.MessageHub$Handler.run(MessageHub.java:404)
	at org.gradle.internal.concurrent.ExecutorPolicy$CatchAndRecordFailures.onExecute(ExecutorPolicy.java:63)
	at org.gradle.internal.concurrent.ManagedExecutorImpl$1.run(ManagedExecutorImpl.java:46)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.gradle.internal.concurrent.ThreadFactoryImpl$ManagedThreadRunnable.run(ThreadFactoryImpl.java:55)
	at java.lang.Thread.run(Thread.java:748)
</failure>
  </testcase>
  <testcase name="runMozillaTest[43, js=testsrc/tests/Rhino_ECMA262_Tests/test_math_asin.js, opt=0]" classname="org.mozilla.javascript.tests.MozillaSuiteTest" time="1.834"/>
  <testcase name="runMozillaTest[44, js=testsrc/tests/Rhino_ECMA262_Tests/test_math_asinh.js, opt=0]" classname="org.mozilla.javascript.tests.MozillaSuiteTest" time="0.182">
    <failure message="junit.framework.AssertionFailedError: junit.framework.AssertionFailedError: junit.framework.AssertionFailedError: In &quot;testsrc/tests/Rhino_ECMA262_Tests/test_math_asinh.js&quot;:&#10;JavaScript errors in testsrc/tests/Rhino_ECMA262_Tests/test_math_asinh.js:&#10;uncaught JavaScript runtime exception: TypeError: Cannot find function asinh in object [object Math].&#10;&#9;at junit.framework.Assert.fail(Assert.java:57)&#10;&#9;at org.mozilla.javascript.tests.MozillaSuiteTest$JunitStatus.failed(MozillaSuiteTest.java:159)&#10;&#9;at org.mozilla.javascript.drivers.ShellTest$Status.hadErrors(ShellTest.java:81)&#10;&#9;at org.mozilla.javascript.drivers.ShellTest$3$1.run(ShellTest.java:292)&#10;&#9;at org.mozilla.javascript.Context.call(Context.java:544)&#10;&#9;at org.mozilla.javascript.ContextFactory.call(ContextFactory.java:515)&#10;&#9;at org.mozilla.javascript.drivers.ShellTest$3.run(ShellTest.java:279)&#10;&#9;at java.lang.Thread.run(Thread.java:748)&#10;&#10;&#9;at junit.framework.Assert.fail(Assert.java:57)&#10;&#9;at org.mozilla.javascript.tests.MozillaSuiteTest$JunitStatus.threw(MozillaSuiteTest.java:175)&#10;&#9;at org.mozilla.javascript.drivers.ShellTest$3$1.run(ShellTest.java:295)&#10;&#9;at org.mozilla.javascript.Context.call(Context.java:544)&#10;&#9;at org.mozilla.javascript.ContextFactory.call(ContextFactory.java:515)&#10;&#9;at org.mozilla.javascript.drivers.ShellTest$3.run(ShellTest.java:279)&#10;&#9;at java.lang.Thread.run(Thread.java:748)&#10;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: junit.framework.AssertionFailedError: junit.framework.AssertionFailedError: In &quot;testsrc/tests/Rhino_ECMA262_Tests/test_math_asinh.js&quot;:
JavaScript errors in testsrc/tests/Rhino_ECMA262_Tests/test_math_asinh.js:
uncaught JavaScript runtime exception: TypeError: Cannot find function asinh in object [object Math].
	at junit.framework.Assert.fail(Assert.java:57)
	at org.mozilla.javascript.tests.MozillaSuiteTest$JunitStatus.failed(MozillaSuiteTest.java:159)
	at org.mozilla.javascript.drivers.ShellTest$Status.hadErrors(ShellTest.java:81)
	at org.mozilla.javascript.drivers.ShellTest$3$1.run(ShellTest.java:292)
	at org.mozilla.javascript.Context.call(Context.java:544)
	at org.mozilla.javascript.ContextFactory.call(ContextFactory.java:515)
	at org.mozilla.javascript.drivers.ShellTest$3.run(ShellTest.java:279)
	at java.lang.Thread.run(Thread.java:748)

	at junit.framework.Assert.fail(Assert.java:57)
	at org.mozilla.javascript.tests.MozillaSuiteTest$JunitStatus.threw(MozillaSuiteTest.java:175)
	at org.mozilla.javascript.drivers.ShellTest$3$1.run(ShellTest.java:295)
	at org.mozilla.javascript.Context.call(Context.java:544)
	at org.mozilla.javascript.ContextFactory.call(ContextFactory.java:515)
	at org.mozilla.javascript.drivers.ShellTest$3.run(ShellTest.java:279)
	at java.lang.Thread.run(Thread.java:748)

	at junit.framework.Assert.fail(Assert.java:57)
	at org.mozilla.javascript.tests.MozillaSuiteTest$JunitStatus.threw(MozillaSuiteTest.java:175)
	at org.mozilla.javascript.drivers.ShellTest.run(ShellTest.java:353)
	at org.mozilla.javascript.tests.MozillaSuiteTest.runMozillaTest(MozillaSuiteTest.java:192)
	at sun.reflect.GeneratedMethodAccessor5.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.junit.runners.Suite.runChild(Suite.java:128)
	at org.junit.runners.Suite.runChild(Suite.java:27)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecuter.runTestClass(JUnitTestClassExecuter.java:114)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecuter.execute(JUnitTestClassExecuter.java:57)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassProcessor.processTestClass(JUnitTestClassProcessor.java:66)
	at org.gradle.api.internal.tasks.testing.SuiteTestClassProcessor.processTestClass(SuiteTestClassProcessor.java:51)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:35)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)
	at org.gradle.internal.dispatch.ContextClassLoaderDispatch.dispatch(ContextClassLoaderDispatch.java:32)
	at org.gradle.internal.dispatch.ProxyDispatchAdapter$DispatchingInvocationHandler.invoke(ProxyDispatchAdapter.java:93)
	at com.sun.proxy.$Proxy1.processTestClass(Unknown Source)
	at org.gradle.api.internal.tasks.testing.worker.TestWorker.processTestClass(TestWorker.java:108)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:35)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)
	at org.gradle.internal.remote.internal.hub.MessageHubBackedObjectConnection$DispatchWrapper.dispatch(MessageHubBackedObjectConnection.java:146)
	at org.gradle.internal.remote.internal.hub.MessageHubBackedObjectConnection$DispatchWrapper.dispatch(MessageHubBackedObjectConnection.java:128)
	at org.gradle.internal.remote.internal.hub.MessageHub$Handler.run(MessageHub.java:404)
	at org.gradle.internal.concurrent.ExecutorPolicy$CatchAndRecordFailures.onExecute(ExecutorPolicy.java:63)
	at org.gradle.internal.concurrent.ManagedExecutorImpl$1.run(ManagedExecutorImpl.java:46)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.gradle.internal.concurrent.ThreadFactoryImpl$ManagedThreadRunnable.run(ThreadFactoryImpl.java:55)
	at java.lang.Thread.run(Thread.java:748)
</failure>
  </testcase>
  <testcase name="runMozillaTest[45, js=testsrc/tests/Rhino_ECMA262_Tests/test_math_atan.js, opt=0]" classname="org.mozilla.javascript.tests.MozillaSuiteTest" time="0.225"/>
  <testcase name="runMozillaTest[46, js=testsrc/tests/Rhino_ECMA262_Tests/test_math_atan2.js, opt=0]" classname="org.mozilla.javascript.tests.MozillaSuiteTest" time="0.263"/>
  <testcase name="runMozillaTest[47, js=testsrc/tests/Rhino_ECMA262_Tests/test_math_atanh.js, opt=0]" classname="org.mozilla.javascript.tests.MozillaSuiteTest" time="0.121">
    <failure message="junit.framework.AssertionFailedError: junit.framework.AssertionFailedError: junit.framework.AssertionFailedError: In &quot;testsrc/tests/Rhino_ECMA262_Tests/test_math_atanh.js&quot;:&#10;JavaScript errors in testsrc/tests/Rhino_ECMA262_Tests/test_math_atanh.js:&#10;uncaught JavaScript runtime exception: TypeError: Cannot find function atanh in object [object Math].&#10;&#9;at junit.framework.Assert.fail(Assert.java:57)&#10;&#9;at org.mozilla.javascript.tests.MozillaSuiteTest$JunitStatus.failed(MozillaSuiteTest.java:159)&#10;&#9;at org.mozilla.javascript.drivers.ShellTest$Status.hadErrors(ShellTest.java:81)&#10;&#9;at org.mozilla.javascript.drivers.ShellTest$3$1.run(ShellTest.java:292)&#10;&#9;at org.mozilla.javascript.Context.call(Context.java:544)&#10;&#9;at org.mozilla.javascript.ContextFactory.call(ContextFactory.java:515)&#10;&#9;at org.mozilla.javascript.drivers.ShellTest$3.run(ShellTest.java:279)&#10;&#9;at java.lang.Thread.run(Thread.java:748)&#10;&#10;&#9;at junit.framework.Assert.fail(Assert.java:57)&#10;&#9;at org.mozilla.javascript.tests.MozillaSuiteTest$JunitStatus.threw(MozillaSuiteTest.java:175)&#10;&#9;at org.mozilla.javascript.drivers.ShellTest$3$1.run(ShellTest.java:295)&#10;&#9;at org.mozilla.javascript.Context.call(Context.java:544)&#10;&#9;at org.mozilla.javascript.ContextFactory.call(ContextFactory.java:515)&#10;&#9;at org.mozilla.javascript.drivers.ShellTest$3.run(ShellTest.java:279)&#10;&#9;at java.lang.Thread.run(Thread.java:748)&#10;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: junit.framework.AssertionFailedError: junit.framework.AssertionFailedError: In &quot;testsrc/tests/Rhino_ECMA262_Tests/test_math_atanh.js&quot;:
JavaScript errors in testsrc/tests/Rhino_ECMA262_Tests/test_math_atanh.js:
uncaught JavaScript runtime exception: TypeError: Cannot find function atanh in object [object Math].
	at junit.framework.Assert.fail(Assert.java:57)
	at org.mozilla.javascript.tests.MozillaSuiteTest$JunitStatus.failed(MozillaSuiteTest.java:159)
	at org.mozilla.javascript.drivers.ShellTest$Status.hadErrors(ShellTest.java:81)
	at org.mozilla.javascript.drivers.ShellTest$3$1.run(ShellTest.java:292)
	at org.mozilla.javascript.Context.call(Context.java:544)
	at org.mozilla.javascript.ContextFactory.call(ContextFactory.java:515)
	at org.mozilla.javascript.drivers.ShellTest$3.run(ShellTest.java:279)
	at java.lang.Thread.run(Thread.java:748)

	at junit.framework.Assert.fail(Assert.java:57)
	at org.mozilla.javascript.tests.MozillaSuiteTest$JunitStatus.threw(MozillaSuiteTest.java:175)
	at org.mozilla.javascript.drivers.ShellTest$3$1.run(ShellTest.java:295)
	at org.mozilla.javascript.Context.call(Context.java:544)
	at org.mozilla.javascript.ContextFactory.call(ContextFactory.java:515)
	at org.mozilla.javascript.drivers.ShellTest$3.run(ShellTest.java:279)
	at java.lang.Thread.run(Thread.java:748)

	at junit.framework.Assert.fail(Assert.java:57)
	at org.mozilla.javascript.tests.MozillaSuiteTest$JunitStatus.threw(MozillaSuiteTest.java:175)
	at org.mozilla.javascript.drivers.ShellTest.run(ShellTest.java:353)
	at org.mozilla.javascript.tests.MozillaSuiteTest.runMozillaTest(MozillaSuiteTest.java:192)
	at sun.reflect.GeneratedMethodAccessor5.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.junit.runners.Suite.runChild(Suite.java:128)
	at org.junit.runners.Suite.runChild(Suite.java:27)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecuter.runTestClass(JUnitTestClassExecuter.java:114)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecuter.execute(JUnitTestClassExecuter.java:57)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassProcessor.processTestClass(JUnitTestClassProcessor.java:66)
	at org.gradle.api.internal.tasks.testing.SuiteTestClassProcessor.processTestClass(SuiteTestClassProcessor.java:51)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:35)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)
	at org.gradle.internal.dispatch.ContextClassLoaderDispatch.dispatch(ContextClassLoaderDispatch.java:32)
	at org.gradle.internal.dispatch.ProxyDispatchAdapter$DispatchingInvocationHandler.invoke(ProxyDispatchAdapter.java:93)
	at com.sun.proxy.$Proxy1.processTestClass(Unknown Source)
	at org.gradle.api.internal.tasks.testing.worker.TestWorker.processTestClass(TestWorker.java:108)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:35)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)
	at org.gradle.internal.remote.internal.hub.MessageHubBackedObjectConnection$DispatchWrapper.dispatch(MessageHubBackedObjectConnection.java:146)
	at org.gradle.internal.remote.internal.hub.MessageHubBackedObjectConnection$DispatchWrapper.dispatch(MessageHubBackedObjectConnection.java:128)
	at org.gradle.internal.remote.internal.hub.MessageHub$Handler.run(MessageHub.java:404)
	at org.gradle.internal.concurrent.ExecutorPolicy$CatchAndRecordFailures.onExecute(ExecutorPolicy.java:63)
	at org.gradle.internal.concurrent.ManagedExecutorImpl$1.run(ManagedExecutorImpl.java:46)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.gradle.internal.concurrent.ThreadFactoryImpl$ManagedThreadRunnable.run(ThreadFactoryImpl.java:55)
	at java.lang.Thread.run(Thread.java:748)
</failure>
  </testcase>
  <testcase name="runMozillaTest[48, js=testsrc/tests/Rhino_ECMA262_Tests/test_math_cbrt.js, opt=0]" classname="org.mozilla.javascript.tests.MozillaSuiteTest" time="0.2"/>
  <testcase name="runMozillaTest[49, js=testsrc/tests/Rhino_ECMA262_Tests/test_math_ceil.js, opt=0]" classname="org.mozilla.javascript.tests.MozillaSuiteTest" time="1.167"/>
  <testcase name="runMozillaTest[50, js=testsrc/tests/Rhino_ECMA262_Tests/test_math_cos.js, opt=0]" classname="org.mozilla.javascript.tests.MozillaSuiteTest" time="0.164"/>
  <testcase name="runMozillaTest[51, js=testsrc/tests/Rhino_ECMA262_Tests/test_math_cosh.js, opt=0]" classname="org.mozilla.javascript.tests.MozillaSuiteTest" time="0.174"/>
  <testcase name="runMozillaTest[52, js=testsrc/tests/Rhino_ECMA262_Tests/test_math_exp.js, opt=0]" classname="org.mozilla.javascript.tests.MozillaSuiteTest" time="0.169"/>
  <testcase name="runMozillaTest[53, js=testsrc/tests/Rhino_ECMA262_Tests/test_math_expm1.js, opt=0]" classname="org.mozilla.javascript.tests.MozillaSuiteTest" time="0.163"/>
  <testcase name="runMozillaTest[54, js=testsrc/tests/Rhino_ECMA262_Tests/test_math_floor.js, opt=0]" classname="org.mozilla.javascript.tests.MozillaSuiteTest" time="0.874"/>
  <testcase name="runMozillaTest[55, js=testsrc/tests/Rhino_ECMA262_Tests/test_math_fround.js, opt=0]" classname="org.mozilla.javascript.tests.MozillaSuiteTest" time="0.151">
    <failure message="junit.framework.AssertionFailedError: junit.framework.AssertionFailedError: junit.framework.AssertionFailedError: In &quot;testsrc/tests/Rhino_ECMA262_Tests/test_math_fround.js&quot;:&#10;JavaScript errors in testsrc/tests/Rhino_ECMA262_Tests/test_math_fround.js:&#10;uncaught JavaScript runtime exception: TypeError: Cannot find function fround in object [object Math].&#10;&#9;at junit.framework.Assert.fail(Assert.java:57)&#10;&#9;at org.mozilla.javascript.tests.MozillaSuiteTest$JunitStatus.failed(MozillaSuiteTest.java:159)&#10;&#9;at org.mozilla.javascript.drivers.ShellTest$Status.hadErrors(ShellTest.java:81)&#10;&#9;at org.mozilla.javascript.drivers.ShellTest$3$1.run(ShellTest.java:292)&#10;&#9;at org.mozilla.javascript.Context.call(Context.java:544)&#10;&#9;at org.mozilla.javascript.ContextFactory.call(ContextFactory.java:515)&#10;&#9;at org.mozilla.javascript.drivers.ShellTest$3.run(ShellTest.java:279)&#10;&#9;at java.lang.Thread.run(Thread.java:748)&#10;&#10;&#9;at junit.framework.Assert.fail(Assert.java:57)&#10;&#9;at org.mozilla.javascript.tests.MozillaSuiteTest$JunitStatus.threw(MozillaSuiteTest.java:175)&#10;&#9;at org.mozilla.javascript.drivers.ShellTest$3$1.run(ShellTest.java:295)&#10;&#9;at org.mozilla.javascript.Context.call(Context.java:544)&#10;&#9;at org.mozilla.javascript.ContextFactory.call(ContextFactory.java:515)&#10;&#9;at org.mozilla.javascript.drivers.ShellTest$3.run(ShellTest.java:279)&#10;&#9;at java.lang.Thread.run(Thread.java:748)&#10;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: junit.framework.AssertionFailedError: junit.framework.AssertionFailedError: In &quot;testsrc/tests/Rhino_ECMA262_Tests/test_math_fround.js&quot;:
JavaScript errors in testsrc/tests/Rhino_ECMA262_Tests/test_math_fround.js:
uncaught JavaScript runtime exception: TypeError: Cannot find function fround in object [object Math].
	at junit.framework.Assert.fail(Assert.java:57)
	at org.mozilla.javascript.tests.MozillaSuiteTest$JunitStatus.failed(MozillaSuiteTest.java:159)
	at org.mozilla.javascript.drivers.ShellTest$Status.hadErrors(ShellTest.java:81)
	at org.mozilla.javascript.drivers.ShellTest$3$1.run(ShellTest.java:292)
	at org.mozilla.javascript.Context.call(Context.java:544)
	at org.mozilla.javascript.ContextFactory.call(ContextFactory.java:515)
	at org.mozilla.javascript.drivers.ShellTest$3.run(ShellTest.java:279)
	at java.lang.Thread.run(Thread.java:748)

	at junit.framework.Assert.fail(Assert.java:57)
	at org.mozilla.javascript.tests.MozillaSuiteTest$JunitStatus.threw(MozillaSuiteTest.java:175)
	at org.mozilla.javascript.drivers.ShellTest$3$1.run(ShellTest.java:295)
	at org.mozilla.javascript.Context.call(Context.java:544)
	at org.mozilla.javascript.ContextFactory.call(ContextFactory.java:515)
	at org.mozilla.javascript.drivers.ShellTest$3.run(ShellTest.java:279)
	at java.lang.Thread.run(Thread.java:748)

	at junit.framework.Assert.fail(Assert.java:57)
	at org.mozilla.javascript.tests.MozillaSuiteTest$JunitStatus.threw(MozillaSuiteTest.java:175)
	at org.mozilla.javascript.drivers.ShellTest.run(ShellTest.java:353)
	at org.mozilla.javascript.tests.MozillaSuiteTest.runMozillaTest(MozillaSuiteTest.java:192)
	at sun.reflect.GeneratedMethodAccessor5.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.junit.runners.Suite.runChild(Suite.java:128)
	at org.junit.runners.Suite.runChild(Suite.java:27)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecuter.runTestClass(JUnitTestClassExecuter.java:114)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecuter.execute(JUnitTestClassExecuter.java:57)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassProcessor.processTestClass(JUnitTestClassProcessor.java:66)
	at org.gradle.api.internal.tasks.testing.SuiteTestClassProcessor.processTestClass(SuiteTestClassProcessor.java:51)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:35)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)
	at org.gradle.internal.dispatch.ContextClassLoaderDispatch.dispatch(ContextClassLoaderDispatch.java:32)
	at org.gradle.internal.dispatch.ProxyDispatchAdapter$DispatchingInvocationHandler.invoke(ProxyDispatchAdapter.java:93)
	at com.sun.proxy.$Proxy1.processTestClass(Unknown Source)
	at org.gradle.api.internal.tasks.testing.worker.TestWorker.processTestClass(TestWorker.java:108)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:35)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)
	at org.gradle.internal.remote.internal.hub.MessageHubBackedObjectConnection$DispatchWrapper.dispatch(MessageHubBackedObjectConnection.java:146)
	at org.gradle.internal.remote.internal.hub.MessageHubBackedObjectConnection$DispatchWrapper.dispatch(MessageHubBackedObjectConnection.java:128)
	at org.gradle.internal.remote.internal.hub.MessageHub$Handler.run(MessageHub.java:404)
	at org.gradle.internal.concurrent.ExecutorPolicy$CatchAndRecordFailures.onExecute(ExecutorPolicy.java:63)
	at org.gradle.internal.concurrent.ManagedExecutorImpl$1.run(ManagedExecutorImpl.java:46)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.gradle.internal.concurrent.ThreadFactoryImpl$ManagedThreadRunnable.run(ThreadFactoryImpl.java:55)
	at java.lang.Thread.run(Thread.java:748)
</failure>
  </testcase>
  <testcase name="runMozillaTest[56, js=testsrc/tests/Rhino_ECMA262_Tests/test_math_log.js, opt=0]" classname="org.mozilla.javascript.tests.MozillaSuiteTest" time="0.605"/>
  <testcase name="runMozillaTest[57, js=testsrc/tests/Rhino_ECMA262_Tests/test_math_log10.js, opt=0]" classname="org.mozilla.javascript.tests.MozillaSuiteTest" time="0.648"/>
  <testcase name="runMozillaTest[58, js=testsrc/tests/Rhino_ECMA262_Tests/test_math_log1p.js, opt=0]" classname="org.mozilla.javascript.tests.MozillaSuiteTest" time="0.687"/>
  <testcase name="runMozillaTest[59, js=testsrc/tests/Rhino_ECMA262_Tests/test_math_log2.js, opt=0]" classname="org.mozilla.javascript.tests.MozillaSuiteTest" time="0.11">
    <failure message="junit.framework.AssertionFailedError: junit.framework.AssertionFailedError: junit.framework.AssertionFailedError: In &quot;testsrc/tests/Rhino_ECMA262_Tests/test_math_log2.js&quot;:&#10;JavaScript errors in testsrc/tests/Rhino_ECMA262_Tests/test_math_log2.js:&#10;uncaught JavaScript runtime exception: TypeError: Cannot find function log2 in object [object Math].&#10;&#9;at junit.framework.Assert.fail(Assert.java:57)&#10;&#9;at org.mozilla.javascript.tests.MozillaSuiteTest$JunitStatus.failed(MozillaSuiteTest.java:159)&#10;&#9;at org.mozilla.javascript.drivers.ShellTest$Status.hadErrors(ShellTest.java:81)&#10;&#9;at org.mozilla.javascript.drivers.ShellTest$3$1.run(ShellTest.java:292)&#10;&#9;at org.mozilla.javascript.Context.call(Context.java:544)&#10;&#9;at org.mozilla.javascript.ContextFactory.call(ContextFactory.java:515)&#10;&#9;at org.mozilla.javascript.drivers.ShellTest$3.run(ShellTest.java:279)&#10;&#9;at java.lang.Thread.run(Thread.java:748)&#10;&#10;&#9;at junit.framework.Assert.fail(Assert.java:57)&#10;&#9;at org.mozilla.javascript.tests.MozillaSuiteTest$JunitStatus.threw(MozillaSuiteTest.java:175)&#10;&#9;at org.mozilla.javascript.drivers.ShellTest$3$1.run(ShellTest.java:295)&#10;&#9;at org.mozilla.javascript.Context.call(Context.java:544)&#10;&#9;at org.mozilla.javascript.ContextFactory.call(ContextFactory.java:515)&#10;&#9;at org.mozilla.javascript.drivers.ShellTest$3.run(ShellTest.java:279)&#10;&#9;at java.lang.Thread.run(Thread.java:748)&#10;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: junit.framework.AssertionFailedError: junit.framework.AssertionFailedError: In &quot;testsrc/tests/Rhino_ECMA262_Tests/test_math_log2.js&quot;:
JavaScript errors in testsrc/tests/Rhino_ECMA262_Tests/test_math_log2.js:
uncaught JavaScript runtime exception: TypeError: Cannot find function log2 in object [object Math].
	at junit.framework.Assert.fail(Assert.java:57)
	at org.mozilla.javascript.tests.MozillaSuiteTest$JunitStatus.failed(MozillaSuiteTest.java:159)
	at org.mozilla.javascript.drivers.ShellTest$Status.hadErrors(ShellTest.java:81)
	at org.mozilla.javascript.drivers.ShellTest$3$1.run(ShellTest.java:292)
	at org.mozilla.javascript.Context.call(Context.java:544)
	at org.mozilla.javascript.ContextFactory.call(ContextFactory.java:515)
	at org.mozilla.javascript.drivers.ShellTest$3.run(ShellTest.java:279)
	at java.lang.Thread.run(Thread.java:748)

	at junit.framework.Assert.fail(Assert.java:57)
	at org.mozilla.javascript.tests.MozillaSuiteTest$JunitStatus.threw(MozillaSuiteTest.java:175)
	at org.mozilla.javascript.drivers.ShellTest$3$1.run(ShellTest.java:295)
	at org.mozilla.javascript.Context.call(Context.java:544)
	at org.mozilla.javascript.ContextFactory.call(ContextFactory.java:515)
	at org.mozilla.javascript.drivers.ShellTest$3.run(ShellTest.java:279)
	at java.lang.Thread.run(Thread.java:748)

	at junit.framework.Assert.fail(Assert.java:57)
	at org.mozilla.javascript.tests.MozillaSuiteTest$JunitStatus.threw(MozillaSuiteTest.java:175)
	at org.mozilla.javascript.drivers.ShellTest.run(ShellTest.java:353)
	at org.mozilla.javascript.tests.MozillaSuiteTest.runMozillaTest(MozillaSuiteTest.java:192)
	at sun.reflect.GeneratedMethodAccessor5.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.junit.runners.Suite.runChild(Suite.java:128)
	at org.junit.runners.Suite.runChild(Suite.java:27)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecuter.runTestClass(JUnitTestClassExecuter.java:114)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecuter.execute(JUnitTestClassExecuter.java:57)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassProcessor.processTestClass(JUnitTestClassProcessor.java:66)
	at org.gradle.api.internal.tasks.testing.SuiteTestClassProcessor.processTestClass(SuiteTestClassProcessor.java:51)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:35)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)
	at org.gradle.internal.dispatch.ContextClassLoaderDispatch.dispatch(ContextClassLoaderDispatch.java:32)
	at org.gradle.internal.dispatch.ProxyDispatchAdapter$DispatchingInvocationHandler.invoke(ProxyDispatchAdapter.java:93)
	at com.sun.proxy.$Proxy1.processTestClass(Unknown Source)
	at org.gradle.api.internal.tasks.testing.worker.TestWorker.processTestClass(TestWorker.java:108)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:35)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)
	at org.gradle.internal.remote.internal.hub.MessageHubBackedObjectConnection$DispatchWrapper.dispatch(MessageHubBackedObjectConnection.java:146)
	at org.gradle.internal.remote.internal.hub.MessageHubBackedObjectConnection$DispatchWrapper.dispatch(MessageHubBackedObjectConnection.java:128)
	at org.gradle.internal.remote.internal.hub.MessageHub$Handler.run(MessageHub.java:404)
	at org.gradle.internal.concurrent.ExecutorPolicy$CatchAndRecordFailures.onExecute(ExecutorPolicy.java:63)
	at org.gradle.internal.concurrent.ManagedExecutorImpl$1.run(ManagedExecutorImpl.java:46)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.gradle.internal.concurrent.ThreadFactoryImpl$ManagedThreadRunnable.run(ThreadFactoryImpl.java:55)
	at java.lang.Thread.run(Thread.java:748)
</failure>
  </testcase>
  <testcase name="runMozillaTest[60, js=testsrc/tests/Rhino_ECMA262_Tests/test_math_round.js, opt=0]" classname="org.mozilla.javascript.tests.MozillaSuiteTest" time="0.144"/>
  <testcase name="runMozillaTest[61, js=testsrc/tests/Rhino_ECMA262_Tests/test_math_sign.js, opt=0]" classname="org.mozilla.javascript.tests.MozillaSuiteTest" time="0.135">
    <failure message="junit.framework.AssertionFailedError: junit.framework.AssertionFailedError: junit.framework.AssertionFailedError: In &quot;testsrc/tests/Rhino_ECMA262_Tests/test_math_sign.js&quot;:&#10;JavaScript errors in testsrc/tests/Rhino_ECMA262_Tests/test_math_sign.js:&#10;uncaught JavaScript runtime exception: TypeError: Cannot find function sign in object [object Math].&#10;&#9;at junit.framework.Assert.fail(Assert.java:57)&#10;&#9;at org.mozilla.javascript.tests.MozillaSuiteTest$JunitStatus.failed(MozillaSuiteTest.java:159)&#10;&#9;at org.mozilla.javascript.drivers.ShellTest$Status.hadErrors(ShellTest.java:81)&#10;&#9;at org.mozilla.javascript.drivers.ShellTest$3$1.run(ShellTest.java:292)&#10;&#9;at org.mozilla.javascript.Context.call(Context.java:544)&#10;&#9;at org.mozilla.javascript.ContextFactory.call(ContextFactory.java:515)&#10;&#9;at org.mozilla.javascript.drivers.ShellTest$3.run(ShellTest.java:279)&#10;&#9;at java.lang.Thread.run(Thread.java:748)&#10;&#10;&#9;at junit.framework.Assert.fail(Assert.java:57)&#10;&#9;at org.mozilla.javascript.tests.MozillaSuiteTest$JunitStatus.threw(MozillaSuiteTest.java:175)&#10;&#9;at org.mozilla.javascript.drivers.ShellTest$3$1.run(ShellTest.java:295)&#10;&#9;at org.mozilla.javascript.Context.call(Context.java:544)&#10;&#9;at org.mozilla.javascript.ContextFactory.call(ContextFactory.java:515)&#10;&#9;at org.mozilla.javascript.drivers.ShellTest$3.run(ShellTest.java:279)&#10;&#9;at java.lang.Thread.run(Thread.java:748)&#10;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: junit.framework.AssertionFailedError: junit.framework.AssertionFailedError: In &quot;testsrc/tests/Rhino_ECMA262_Tests/test_math_sign.js&quot;:
JavaScript errors in testsrc/tests/Rhino_ECMA262_Tests/test_math_sign.js:
uncaught JavaScript runtime exception: TypeError: Cannot find function sign in object [object Math].
	at junit.framework.Assert.fail(Assert.java:57)
	at org.mozilla.javascript.tests.MozillaSuiteTest$JunitStatus.failed(MozillaSuiteTest.java:159)
	at org.mozilla.javascript.drivers.ShellTest$Status.hadErrors(ShellTest.java:81)
	at org.mozilla.javascript.drivers.ShellTest$3$1.run(ShellTest.java:292)
	at org.mozilla.javascript.Context.call(Context.java:544)
	at org.mozilla.javascript.ContextFactory.call(ContextFactory.java:515)
	at org.mozilla.javascript.drivers.ShellTest$3.run(ShellTest.java:279)
	at java.lang.Thread.run(Thread.java:748)

	at junit.framework.Assert.fail(Assert.java:57)
	at org.mozilla.javascript.tests.MozillaSuiteTest$JunitStatus.threw(MozillaSuiteTest.java:175)
	at org.mozilla.javascript.drivers.ShellTest$3$1.run(ShellTest.java:295)
	at org.mozilla.javascript.Context.call(Context.java:544)
	at org.mozilla.javascript.ContextFactory.call(ContextFactory.java:515)
	at org.mozilla.javascript.drivers.ShellTest$3.run(ShellTest.java:279)
	at java.lang.Thread.run(Thread.java:748)

	at junit.framework.Assert.fail(Assert.java:57)
	at org.mozilla.javascript.tests.MozillaSuiteTest$JunitStatus.threw(MozillaSuiteTest.java:175)
	at org.mozilla.javascript.drivers.ShellTest.run(ShellTest.java:353)
	at org.mozilla.javascript.tests.MozillaSuiteTest.runMozillaTest(MozillaSuiteTest.java:192)
	at sun.reflect.GeneratedMethodAccessor5.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.junit.runners.Suite.runChild(Suite.java:128)
	at org.junit.runners.Suite.runChild(Suite.java:27)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecuter.runTestClass(JUnitTestClassExecuter.java:114)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecuter.execute(JUnitTestClassExecuter.java:57)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassProcessor.processTestClass(JUnitTestClassProcessor.java:66)
	at org.gradle.api.internal.tasks.testing.SuiteTestClassProcessor.processTestClass(SuiteTestClassProcessor.java:51)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:35)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)
	at org.gradle.internal.dispatch.ContextClassLoaderDispatch.dispatch(ContextClassLoaderDispatch.java:32)
	at org.gradle.internal.dispatch.ProxyDispatchAdapter$DispatchingInvocationHandler.invoke(ProxyDispatchAdapter.java:93)
	at com.sun.proxy.$Proxy1.processTestClass(Unknown Source)
	at org.gradle.api.internal.tasks.testing.worker.TestWorker.processTestClass(TestWorker.java:108)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:35)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)
	at org.gradle.internal.remote.internal.hub.MessageHubBackedObjectConnection$DispatchWrapper.dispatch(MessageHubBackedObjectConnection.java:146)
	at org.gradle.internal.remote.internal.hub.MessageHubBackedObjectConnection$DispatchWrapper.dispatch(MessageHubBackedObjectConnection.java:128)
	at org.gradle.internal.remote.internal.hub.MessageHub$Handler.run(MessageHub.java:404)
	at org.gradle.internal.concurrent.ExecutorPolicy$CatchAndRecordFailures.onExecute(ExecutorPolicy.java:63)
	at org.gradle.internal.concurrent.ManagedExecutorImpl$1.run(ManagedExecutorImpl.java:46)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.gradle.internal.concurrent.ThreadFactoryImpl$ManagedThreadRunnable.run(ThreadFactoryImpl.java:55)
	at java.lang.Thread.run(Thread.java:748)
</failure>
  </testcase>
  <testcase name="runMozillaTest[62, js=testsrc/tests/Rhino_ECMA262_Tests/test_math_sin.js, opt=0]" classname="org.mozilla.javascript.tests.MozillaSuiteTest" time="0.182"/>
  <testcase name="runMozillaTest[63, js=testsrc/tests/Rhino_ECMA262_Tests/test_math_sinh.js, opt=0]" classname="org.mozilla.javascript.tests.MozillaSuiteTest" time="0.142"/>
  <testcase name="runMozillaTest[64, js=testsrc/tests/Rhino_ECMA262_Tests/test_math_sqrt.js, opt=0]" classname="org.mozilla.javascript.tests.MozillaSuiteTest" time="0.525"/>
  <testcase name="runMozillaTest[65, js=testsrc/tests/Rhino_ECMA262_Tests/test_math_tan.js, opt=0]" classname="org.mozilla.javascript.tests.MozillaSuiteTest" time="0.168"/>
  <testcase name="runMozillaTest[66, js=testsrc/tests/Rhino_ECMA262_Tests/test_math_tanh.js, opt=0]" classname="org.mozilla.javascript.tests.MozillaSuiteTest" time="0.118"/>
  <testcase name="runMozillaTest[67, js=testsrc/tests/Rhino_ECMA262_Tests/test_math_trunc.js, opt=0]" classname="org.mozilla.javascript.tests.MozillaSuiteTest" time="0.174"/>
  <testcase name="runMozillaTest[68, js=testsrc/tests/Rhino_ECMA262_Tests/test_number_isfinite.js, opt=0]" classname="org.mozilla.javascript.tests.MozillaSuiteTest" time="1.217"/>
  <testcase name="runMozillaTest[69, js=testsrc/tests/Rhino_ECMA262_Tests/test_number_isinteger.js, opt=0]" classname="org.mozilla.javascript.tests.MozillaSuiteTest" time="0.844"/>
  <testcase name="runMozillaTest[70, js=testsrc/tests/Rhino_ECMA262_Tests/test_number_isnan.js, opt=0]" classname="org.mozilla.javascript.tests.MozillaSuiteTest" time="0.821"/>
  <testcase name="runMozillaTest[71, js=testsrc/tests/Rhino_ECMA262_Tests/test_number_issafeinteger.js, opt=0]" classname="org.mozilla.javascript.tests.MozillaSuiteTest" time="1.231"/>
  <testcase name="runMozillaTest[72, js=testsrc/tests/Rhino_ECMA262_Tests/test_number_prototype_toexponential.js, opt=0]" classname="org.mozilla.javascript.tests.MozillaSuiteTest" time="0.935"/>
  <testcase name="runMozillaTest[73, js=testsrc/tests/Rhino_ECMA262_Tests/test_number_prototype_toprecision.js, opt=0]" classname="org.mozilla.javascript.tests.MozillaSuiteTest" time="0.998"/>
  <testcase name="runMozillaTest[74, js=testsrc/tests/Rhino_ECMA262_Tests/test_string_prototype_charat.js, opt=0]" classname="org.mozilla.javascript.tests.MozillaSuiteTest" time="0.65"/>
  <testcase name="runMozillaTest[75, js=testsrc/tests/Rhino_ECMA262_Tests/test_string_prototype_charcodeat.js, opt=0]" classname="org.mozilla.javascript.tests.MozillaSuiteTest" time="0.652"/>
  <testcase name="runMozillaTest[76, js=testsrc/tests/Rhino_ECMA262_Tests/test_string_prototype_codepointat.js, opt=0]" classname="org.mozilla.javascript.tests.MozillaSuiteTest" time="0.651"/>
  <testcase name="runMozillaTest[77, js=testsrc/tests/Rhino_ECMA262_Tests/test_string_prototype_endswith.js, opt=0]" classname="org.mozilla.javascript.tests.MozillaSuiteTest" time="0.367"/>
  <testcase name="runMozillaTest[78, js=testsrc/tests/Rhino_ECMA262_Tests/test_string_prototype_includes.js, opt=0]" classname="org.mozilla.javascript.tests.MozillaSuiteTest" time="0.325"/>
  <testcase name="runMozillaTest[79, js=testsrc/tests/Rhino_ECMA262_Tests/test_string_prototype_padend.js, opt=0]" classname="org.mozilla.javascript.tests.MozillaSuiteTest" time="0.218">
    <failure message="junit.framework.AssertionFailedError: junit.framework.AssertionFailedError: junit.framework.AssertionFailedError: In &quot;testsrc/tests/Rhino_ECMA262_Tests/test_string_prototype_padend.js&quot;:&#10;JavaScript errors in testsrc/tests/Rhino_ECMA262_Tests/test_string_prototype_padend.js:&#10;uncaught JavaScript runtime exception: TypeError: Cannot find function padEnd in object PW12.&#10;&#9;at junit.framework.Assert.fail(Assert.java:57)&#10;&#9;at org.mozilla.javascript.tests.MozillaSuiteTest$JunitStatus.failed(MozillaSuiteTest.java:159)&#10;&#9;at org.mozilla.javascript.drivers.ShellTest$Status.hadErrors(ShellTest.java:81)&#10;&#9;at org.mozilla.javascript.drivers.ShellTest$3$1.run(ShellTest.java:292)&#10;&#9;at org.mozilla.javascript.Context.call(Context.java:544)&#10;&#9;at org.mozilla.javascript.ContextFactory.call(ContextFactory.java:515)&#10;&#9;at org.mozilla.javascript.drivers.ShellTest$3.run(ShellTest.java:279)&#10;&#9;at java.lang.Thread.run(Thread.java:748)&#10;&#10;&#9;at junit.framework.Assert.fail(Assert.java:57)&#10;&#9;at org.mozilla.javascript.tests.MozillaSuiteTest$JunitStatus.threw(MozillaSuiteTest.java:175)&#10;&#9;at org.mozilla.javascript.drivers.ShellTest$3$1.run(ShellTest.java:295)&#10;&#9;at org.mozilla.javascript.Context.call(Context.java:544)&#10;&#9;at org.mozilla.javascript.ContextFactory.call(ContextFactory.java:515)&#10;&#9;at org.mozilla.javascript.drivers.ShellTest$3.run(ShellTest.java:279)&#10;&#9;at java.lang.Thread.run(Thread.java:748)&#10;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: junit.framework.AssertionFailedError: junit.framework.AssertionFailedError: In &quot;testsrc/tests/Rhino_ECMA262_Tests/test_string_prototype_padend.js&quot;:
JavaScript errors in testsrc/tests/Rhino_ECMA262_Tests/test_string_prototype_padend.js:
uncaught JavaScript runtime exception: TypeError: Cannot find function padEnd in object PW12.
	at junit.framework.Assert.fail(Assert.java:57)
	at org.mozilla.javascript.tests.MozillaSuiteTest$JunitStatus.failed(MozillaSuiteTest.java:159)
	at org.mozilla.javascript.drivers.ShellTest$Status.hadErrors(ShellTest.java:81)
	at org.mozilla.javascript.drivers.ShellTest$3$1.run(ShellTest.java:292)
	at org.mozilla.javascript.Context.call(Context.java:544)
	at org.mozilla.javascript.ContextFactory.call(ContextFactory.java:515)
	at org.mozilla.javascript.drivers.ShellTest$3.run(ShellTest.java:279)
	at java.lang.Thread.run(Thread.java:748)

	at junit.framework.Assert.fail(Assert.java:57)
	at org.mozilla.javascript.tests.MozillaSuiteTest$JunitStatus.threw(MozillaSuiteTest.java:175)
	at org.mozilla.javascript.drivers.ShellTest$3$1.run(ShellTest.java:295)
	at org.mozilla.javascript.Context.call(Context.java:544)
	at org.mozilla.javascript.ContextFactory.call(ContextFactory.java:515)
	at org.mozilla.javascript.drivers.ShellTest$3.run(ShellTest.java:279)
	at java.lang.Thread.run(Thread.java:748)

	at junit.framework.Assert.fail(Assert.java:57)
	at org.mozilla.javascript.tests.MozillaSuiteTest$JunitStatus.threw(MozillaSuiteTest.java:175)
	at org.mozilla.javascript.drivers.ShellTest.run(ShellTest.java:353)
	at org.mozilla.javascript.tests.MozillaSuiteTest.runMozillaTest(MozillaSuiteTest.java:192)
	at sun.reflect.GeneratedMethodAccessor5.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.junit.runners.Suite.runChild(Suite.java:128)
	at org.junit.runners.Suite.runChild(Suite.java:27)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecuter.runTestClass(JUnitTestClassExecuter.java:114)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecuter.execute(JUnitTestClassExecuter.java:57)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassProcessor.processTestClass(JUnitTestClassProcessor.java:66)
	at org.gradle.api.internal.tasks.testing.SuiteTestClassProcessor.processTestClass(SuiteTestClassProcessor.java:51)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:35)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)
	at org.gradle.internal.dispatch.ContextClassLoaderDispatch.dispatch(ContextClassLoaderDispatch.java:32)
	at org.gradle.internal.dispatch.ProxyDispatchAdapter$DispatchingInvocationHandler.invoke(ProxyDispatchAdapter.java:93)
	at com.sun.proxy.$Proxy1.processTestClass(Unknown Source)
	at org.gradle.api.internal.tasks.testing.worker.TestWorker.processTestClass(TestWorker.java:108)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:35)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)
	at org.gradle.internal.remote.internal.hub.MessageHubBackedObjectConnection$DispatchWrapper.dispatch(MessageHubBackedObjectConnection.java:146)
	at org.gradle.internal.remote.internal.hub.MessageHubBackedObjectConnection$DispatchWrapper.dispatch(MessageHubBackedObjectConnection.java:128)
	at org.gradle.internal.remote.internal.hub.MessageHub$Handler.run(MessageHub.java:404)
	at org.gradle.internal.concurrent.ExecutorPolicy$CatchAndRecordFailures.onExecute(ExecutorPolicy.java:63)
	at org.gradle.internal.concurrent.ManagedExecutorImpl$1.run(ManagedExecutorImpl.java:46)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.gradle.internal.concurrent.ThreadFactoryImpl$ManagedThreadRunnable.run(ThreadFactoryImpl.java:55)
	at java.lang.Thread.run(Thread.java:748)
</failure>
  </testcase>
  <testcase name="runMozillaTest[80, js=testsrc/tests/Rhino_ECMA262_Tests/test_string_prototype_padstart.js, opt=0]" classname="org.mozilla.javascript.tests.MozillaSuiteTest" time="0.243">
    <failure message="junit.framework.AssertionFailedError: junit.framework.AssertionFailedError: junit.framework.AssertionFailedError: In &quot;testsrc/tests/Rhino_ECMA262_Tests/test_string_prototype_padstart.js&quot;:&#10;JavaScript errors in testsrc/tests/Rhino_ECMA262_Tests/test_string_prototype_padstart.js:&#10;uncaught JavaScript runtime exception: TypeError: Cannot find function padStart in object KU4QOJ6.&#10;&#9;at junit.framework.Assert.fail(Assert.java:57)&#10;&#9;at org.mozilla.javascript.tests.MozillaSuiteTest$JunitStatus.failed(MozillaSuiteTest.java:159)&#10;&#9;at org.mozilla.javascript.drivers.ShellTest$Status.hadErrors(ShellTest.java:81)&#10;&#9;at org.mozilla.javascript.drivers.ShellTest$3$1.run(ShellTest.java:292)&#10;&#9;at org.mozilla.javascript.Context.call(Context.java:544)&#10;&#9;at org.mozilla.javascript.ContextFactory.call(ContextFactory.java:515)&#10;&#9;at org.mozilla.javascript.drivers.ShellTest$3.run(ShellTest.java:279)&#10;&#9;at java.lang.Thread.run(Thread.java:748)&#10;&#10;&#9;at junit.framework.Assert.fail(Assert.java:57)&#10;&#9;at org.mozilla.javascript.tests.MozillaSuiteTest$JunitStatus.threw(MozillaSuiteTest.java:175)&#10;&#9;at org.mozilla.javascript.drivers.ShellTest$3$1.run(ShellTest.java:295)&#10;&#9;at org.mozilla.javascript.Context.call(Context.java:544)&#10;&#9;at org.mozilla.javascript.ContextFactory.call(ContextFactory.java:515)&#10;&#9;at org.mozilla.javascript.drivers.ShellTest$3.run(ShellTest.java:279)&#10;&#9;at java.lang.Thread.run(Thread.java:748)&#10;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: junit.framework.AssertionFailedError: junit.framework.AssertionFailedError: In &quot;testsrc/tests/Rhino_ECMA262_Tests/test_string_prototype_padstart.js&quot;:
JavaScript errors in testsrc/tests/Rhino_ECMA262_Tests/test_string_prototype_padstart.js:
uncaught JavaScript runtime exception: TypeError: Cannot find function padStart in object KU4QOJ6.
	at junit.framework.Assert.fail(Assert.java:57)
	at org.mozilla.javascript.tests.MozillaSuiteTest$JunitStatus.failed(MozillaSuiteTest.java:159)
	at org.mozilla.javascript.drivers.ShellTest$Status.hadErrors(ShellTest.java:81)
	at org.mozilla.javascript.drivers.ShellTest$3$1.run(ShellTest.java:292)
	at org.mozilla.javascript.Context.call(Context.java:544)
	at org.mozilla.javascript.ContextFactory.call(ContextFactory.java:515)
	at org.mozilla.javascript.drivers.ShellTest$3.run(ShellTest.java:279)
	at java.lang.Thread.run(Thread.java:748)

	at junit.framework.Assert.fail(Assert.java:57)
	at org.mozilla.javascript.tests.MozillaSuiteTest$JunitStatus.threw(MozillaSuiteTest.java:175)
	at org.mozilla.javascript.drivers.ShellTest$3$1.run(ShellTest.java:295)
	at org.mozilla.javascript.Context.call(Context.java:544)
	at org.mozilla.javascript.ContextFactory.call(ContextFactory.java:515)
	at org.mozilla.javascript.drivers.ShellTest$3.run(ShellTest.java:279)
	at java.lang.Thread.run(Thread.java:748)

	at junit.framework.Assert.fail(Assert.java:57)
	at org.mozilla.javascript.tests.MozillaSuiteTest$JunitStatus.threw(MozillaSuiteTest.java:175)
	at org.mozilla.javascript.drivers.ShellTest.run(ShellTest.java:353)
	at org.mozilla.javascript.tests.MozillaSuiteTest.runMozillaTest(MozillaSuiteTest.java:192)
	at sun.reflect.GeneratedMethodAccessor5.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.junit.runners.Suite.runChild(Suite.java:128)
	at org.junit.runners.Suite.runChild(Suite.java:27)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecuter.runTestClass(JUnitTestClassExecuter.java:114)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecuter.execute(JUnitTestClassExecuter.java:57)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassProcessor.processTestClass(JUnitTestClassProcessor.java:66)
	at org.gradle.api.internal.tasks.testing.SuiteTestClassProcessor.processTestClass(SuiteTestClassProcessor.java:51)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:35)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)
	at org.gradle.internal.dispatch.ContextClassLoaderDispatch.dispatch(ContextClassLoaderDispatch.java:32)
	at org.gradle.internal.dispatch.ProxyDispatchAdapter$DispatchingInvocationHandler.invoke(ProxyDispatchAdapter.java:93)
	at com.sun.proxy.$Proxy1.processTestClass(Unknown Source)
	at org.gradle.api.internal.tasks.testing.worker.TestWorker.processTestClass(TestWorker.java:108)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:35)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)
	at org.gradle.internal.remote.internal.hub.MessageHubBackedObjectConnection$DispatchWrapper.dispatch(MessageHubBackedObjectConnection.java:146)
	at org.gradle.internal.remote.internal.hub.MessageHubBackedObjectConnection$DispatchWrapper.dispatch(MessageHubBackedObjectConnection.java:128)
	at org.gradle.internal.remote.internal.hub.MessageHub$Handler.run(MessageHub.java:404)
	at org.gradle.internal.concurrent.ExecutorPolicy$CatchAndRecordFailures.onExecute(ExecutorPolicy.java:63)
	at org.gradle.internal.concurrent.ManagedExecutorImpl$1.run(ManagedExecutorImpl.java:46)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.gradle.internal.concurrent.ThreadFactoryImpl$ManagedThreadRunnable.run(ThreadFactoryImpl.java:55)
	at java.lang.Thread.run(Thread.java:748)
</failure>
  </testcase>
  <testcase name="runMozillaTest[81, js=testsrc/tests/Rhino_ECMA262_Tests/test_string_prototype_repeat.js, opt=0]" classname="org.mozilla.javascript.tests.MozillaSuiteTest" time="0.484"/>
  <testcase name="runMozillaTest[82, js=testsrc/tests/Rhino_ECMA262_Tests/test_string_prototype_startswith.js, opt=0]" classname="org.mozilla.javascript.tests.MozillaSuiteTest" time="2.538"/>
  <system-out><![CDATA[In "testsrc/tests/Rhino_ECMA262_Tests/test_array_prototype_includes.js":
JavaScript errors in testsrc/tests/Rhino_ECMA262_Tests/test_array_prototype_includes.js:
uncaught JavaScript runtime exception: TypeError: Cannot find function includes in object /\r\n|\r|\n/.
In "testsrc/tests/Rhino_ECMA262_Tests/test_map_prototype_clear.js":
JavaScript errors in testsrc/tests/Rhino_ECMA262_Tests/test_map_prototype_clear.js:
uncaught JavaScript runtime exception: ReferenceError: "Map" is not defined.
In "testsrc/tests/Rhino_ECMA262_Tests/test_map_prototype_delete.js":
JavaScript errors in testsrc/tests/Rhino_ECMA262_Tests/test_map_prototype_delete.js:
testsrc/tests/Rhino_ECMA262_Tests/test_map_prototype_delete.js:120: missing name after . operator
			var output = randominput.delete(key);
			                               ^
testsrc/tests/Rhino_ECMA262_Tests/test_map_prototype_delete.js:130: missing name after . operator
			var output = randominput.delete(key);
			                               ^
testsrc/tests/Rhino_ECMA262_Tests/test_map_prototype_delete.js:1: Compilation produced 2 syntax errors.
In "testsrc/tests/Rhino_ECMA262_Tests/test_map_prototype_foreach.js":
JavaScript errors in testsrc/tests/Rhino_ECMA262_Tests/test_map_prototype_foreach.js:
uncaught JavaScript runtime exception: ReferenceError: "Map" is not defined.
In "testsrc/tests/Rhino_ECMA262_Tests/test_map_prototype_get.js":
JavaScript errors in testsrc/tests/Rhino_ECMA262_Tests/test_map_prototype_get.js:
uncaught JavaScript runtime exception: ReferenceError: "Map" is not defined.
In "testsrc/tests/Rhino_ECMA262_Tests/test_map_prototype_has.js":
JavaScript errors in testsrc/tests/Rhino_ECMA262_Tests/test_map_prototype_has.js:
uncaught JavaScript runtime exception: ReferenceError: "Map" is not defined.
In "testsrc/tests/Rhino_ECMA262_Tests/test_map_prototype_set.js":
JavaScript errors in testsrc/tests/Rhino_ECMA262_Tests/test_map_prototype_set.js:
uncaught JavaScript runtime exception: ReferenceError: "Map" is not defined.
In "testsrc/tests/Rhino_ECMA262_Tests/test_math_acosh.js":
JavaScript errors in testsrc/tests/Rhino_ECMA262_Tests/test_math_acosh.js:
uncaught JavaScript runtime exception: TypeError: Cannot find function acosh in object [object Math].
In "testsrc/tests/Rhino_ECMA262_Tests/test_math_asinh.js":
JavaScript errors in testsrc/tests/Rhino_ECMA262_Tests/test_math_asinh.js:
uncaught JavaScript runtime exception: TypeError: Cannot find function asinh in object [object Math].
In "testsrc/tests/Rhino_ECMA262_Tests/test_math_atanh.js":
JavaScript errors in testsrc/tests/Rhino_ECMA262_Tests/test_math_atanh.js:
uncaught JavaScript runtime exception: TypeError: Cannot find function atanh in object [object Math].
In "testsrc/tests/Rhino_ECMA262_Tests/test_math_fround.js":
JavaScript errors in testsrc/tests/Rhino_ECMA262_Tests/test_math_fround.js:
uncaught JavaScript runtime exception: TypeError: Cannot find function fround in object [object Math].
In "testsrc/tests/Rhino_ECMA262_Tests/test_math_log2.js":
JavaScript errors in testsrc/tests/Rhino_ECMA262_Tests/test_math_log2.js:
uncaught JavaScript runtime exception: TypeError: Cannot find function log2 in object [object Math].
In "testsrc/tests/Rhino_ECMA262_Tests/test_math_sign.js":
JavaScript errors in testsrc/tests/Rhino_ECMA262_Tests/test_math_sign.js:
uncaught JavaScript runtime exception: TypeError: Cannot find function sign in object [object Math].
In "testsrc/tests/Rhino_ECMA262_Tests/test_string_prototype_padend.js":
JavaScript errors in testsrc/tests/Rhino_ECMA262_Tests/test_string_prototype_padend.js:
uncaught JavaScript runtime exception: TypeError: Cannot find function padEnd in object PW12.
In "testsrc/tests/Rhino_ECMA262_Tests/test_string_prototype_padstart.js":
JavaScript errors in testsrc/tests/Rhino_ECMA262_Tests/test_string_prototype_padstart.js:
uncaught JavaScript runtime exception: TypeError: Cannot find function padStart in object KU4QOJ6.
]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
